<!DOCTYPE HTML PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html lang="en" xmlns="http://www.w3.org/1999/xhtml" xmlns:foaf="http://xmlns.com/foaf/0.1/" xmlns:dc="http://purl.org/dc/terms/" xmlns:doi="http://dx.doi.org/" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#" xml:lang="en">

<!DOCTYPE HTML PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">


<meta name="wpd_version" content="0.2">
<meta name="wpd_baseurl" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0014758">
<meta name="wpd_url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0014758">
<meta name="wpd_date" content="2011-02-13T11:10Z">






<script type="text/javascript">var _sf_startpt=(new Date()).getTime()</script>
   <title>PLoS ONE: Effect of Xpcl1 Activation and p27Kip1 Loss on Gene Expression in Murine Lymphoma</title>




<meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7">

<link rel="shortcut icon" href="about:blank?favicon.ico" type="image/x-icon">
<link rel="home" title="home" href="http://www.plosone.org/home.action">


<link rel="alternate" type="application/atom+xml" title="PLoS ONE: New Articles" href="http://www.plosone.org/article/feed">


<!-- global_css.ftl -->








<meta name="description" content="PLoS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world.">

<meta name="keywords" content="PLoS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering">


<!--
<rdf:RDF xmlns="http://web.resource.org/cc/"
   xmlns:dc="http://purl.org/dc/elements/1.1/"
   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
<Work rdf:about="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0014758?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0014758">
   <license rdf:resource="http://creativecommons.org/licenses/by/2.5/" />
</Work>
<License rdf:about="http://creativecommons.org/licenses/by/2.5/">
   <permits rdf:resource="http://web.resource.org/cc/Reproduction" />
   <permits rdf:resource="http://web.resource.org/cc/Distribution" />
   <requires rdf:resource="http://web.resource.org/cc/Notice" />
   <requires rdf:resource="http://web.resource.org/cc/Attribution" />
   <permits rdf:resource="http://web.resource.org/cc/DerivativeWorks" />
</License>
<rdf:Description
     rdf:about="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0014758?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0014758"
     dc:identifier="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0014758?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0014758"
     dc:title="PLoS ONE: Effect of Xpcl1 Activation and p27Kip1 Loss on Gene Expression in Murine Lymphoma"
       trackback:ping="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0014758/trackback"
     />
</rdf:RDF>

-->

<link rel="stylesheet" type="text/css" href="infodoi10.1371journal.pone.css" media="all">
</head>
<body>
<span property="dc:language" content="en"></span>
<!-- begin : container -->
<div id="container">
  <!-- begin : top banner external ad space -->
  <div id="topBanner">


<!-- begin : left banner slot -->
<div class="left">
  <iframe id="aa97ff20" name="aa97ff20" src="infodoi10.1371journal.pone_1.html" frameborder="0" height="60" scrolling="no" width="468">&lt;a
    href='http://ads.plos.org/www/delivery/ck.php?n=aa97ff20&amp;cb=5725'
    target='_top'&gt;&lt;img src='http://ads.plos.org/www/delivery/avw.php?zoneid=287&amp;cb=32&amp;n=aa97ff20'
    border='0' alt='' /&gt;&lt;/a&gt;</iframe>
</div>
<!-- end : left banner slot -->
<!-- begin : right banner slot -->
<div class="right">
  <iframe id="acfd0f5a" name="acfd0f5a" src="infodoi10.1371journal.pone_2.html" frameborder="0" height="60" scrolling="no" width="468">&lt;a
    href='http://ads.plos.org/www/delivery/ck.php?n=acfd0f5a&amp;cb=2257'
    target='_top'&gt;&lt;img src='http://ads.plos.org/www/delivery/avw.php?zoneid=40&amp;cb=3667&amp;n=acfd0f5a'
    border='0' alt='' /&gt;&lt;/a&gt;</iframe>
</div>
<!-- end : right banner slot -->
  </div>
  <!-- end : top banner external ad space -->

  <!-- begin : header -->
  <div id="hdr" class="login">

  <!-- begin : logo -->
  <div id="logo" title="PLoS ONE"><a href="http://www.plosone.org/home.action" title="PLoS ONE"><span>PLoS ONE</span></a></div>
  <!-- end : logo -->
  <!-- begin : user controls -->





  <div id="user">
    <div>
      <ul>
        <li><a href="http://www.plosone.org/user/secure/secureRedirect.action?goTo=%2Farticle%2Finfo%3Adoi%2F10.1371%2Fjournal.pone.0014758" class="feedback"><strong>Login</strong></a> | </li>
        <li><a href="https://register.plos.org/ambra-registration/register.action">Create Account</a> | </li>
        <li class="feedback"><a href="http://www.plosone.org/feedbackCreate.action?page=%252Farticle%252Finfo%253Adoi%252F10.1371%252Fjournal.pone.0014758" title="Send us your feedback">Feedback</a></li>
      </ul>
    </div>
  </div>


  <!-- end : user controls -->
  <!-- begin search links -->
  <ul id="links"><li class="browse"><a href="http://www.plosone.org/article/browse.action?field=date" title="Browse Articles">Browse</a></li><li class="rss"><a href="http://www.plosone.org/static/rssFeeds.action" title="RSS Feeds">RSS</a></li></ul>
  <!-- end : search links -->
  <!-- begin : dashboard -->
  <div id="db">
    <form name="searchForm" action="http://www.plosone.org/search/simpleSearch.action" method="get">
      <input name="from" value="globalSimpleSearch" type="hidden">
      <input name="filterJournals" value="PLoSONE" type="hidden">
      <fieldset>
        <legend>Search</legend>
        <label for="search">Search</label>
        <div class="wrap"><input id="search" name="query" value="Search articles..." onfocus="if(this.value=='Search articles...')value='';" onblur="if(this.value=='')value='Search articles...';" class="searchField" alt="Search articles..." type="text"></div>
        <input src="search_btn1.gif" onclick="submit();" value="ftsearch" alt="SEARCH" tabindex="3" class="button" type="image">
      </fieldset>
    </form>
    <form name="gasf" action="http://www.plosone.org/search/advancedSearch.action" method="get">
<input name="noSearchFlag" value="true" id="noSearchFlag" type="hidden"><input name="query" value="" id="query" type="hidden">    </form>
    <a id="advSearch" href="#" onclick="if(document.searchForm.query.value!='Search articles...')document.gasf.query.value=document.searchForm.query.value;document.gasf.submit();return false;">Advanced Search</a>
  </div>
  <!-- end : dashboard -->
  <!-- begin : navigation -->
  <ul id="nav">
    <li><a href="http://www.plosone.org/home.action" tabindex="101">Home</a></li>
    <li><a href="http://www.plosone.org/static/browse.action" tabindex="102">Browse Articles</a>
        <ul>
           <li><a href="http://www.plosone.org/article/browse.action?field=date">By Publication Date</a></li>
          <li><a href="http://www.plosone.org/article/browse.action">By Subject</a></li>
         <li><a href="http://www.ploscollections.org/static/poneCollections.action">Collections</a></li>
       </ul>
    </li>
    <li><a href="http://www.plosone.org/static/about.action" tabindex="103">About</a>
        <ul>
          <li><a href="http://www.plosone.org/static/information.action">Journal Information</a></li>
          <li><a href="http://www.plosone.org/static/edboard.action">Editorial Board</a></li>
          <li><a href="http://www.plosone.org/static/almInfo.action">Article-Level Metrics</a></li>
          <li><a href="http://www.plosone.org/static/license.action">Open-Access License</a></li>
          <li><a href="http://www.plosone.org/static/contact.action">Contact Us</a></li>
        </ul>
      </li>
    <li><a href="http://www.plosone.org/static/users.action" tabindex="104">For Readers</a>
        <ul>
          <li><a href="http://www.plosone.org/static/commentGuidelines.action">Guidelines for Notes, Comments, and Corrections</a></li>
          <li><a href="http://www.plosone.org/static/ratingGuidelines.action">Guidelines for Rating</a></li>
          <li><a href="http://www.plosone.org/static/help.action">Help Using this Site</a></li>
		  <li><a href="http://www.plosone.org/static/journalClub.action">Journal Club Archives</a></li>
          <li><a href="http://www.plosone.org/static/sitemap.action">Site Map</a></li>
        </ul>
      </li>
    <li><a href="http://www.plosone.org/static/authors.action" tabindex="105">For Authors and Reviewers</a>
        <ul>
          <li><a href="http://www.plosone.org/static/whypublish.action">Why Publish With Us?</a></li>
          <li><a href="http://www.plosone.org/static/review.action">Editorial and Peer-Review Process</a></li>
          <li><a href="http://www.plosone.org/static/policies.action">Editorial and Publishing Policies</a></li>
          <li><a href="http://www.plosone.org/static/guidelines.action">Author Guidelines</a></li>
          <li><a href="http://www.plosone.org/static/figureGuidelines.action">Figure and Table Guidelines</a></li>
          <li><a href="http://www.plosone.org/static/checklist.action">Submit Your Paper</a></li>
          <li><a href="http://www.plosone.org/static/reviewerGuidelines.action">Reviewer Guidelines</a></li>
	  <li><a href="http://www.plosone.org/static/peerReviewers.action">Thanking All Peer Reviewers</a></li>
        </ul>
      </li>
      <li class="journalnav"><a href="http://www.plos.org/" title="Public Library of Science" tabindex="110" class="drop">PLoS.org</a>
        <ul>
          <li><a href="http://www.plos.org/oa/index.html" title="Open Access Statement">Open Access</a></li>
          <li><a href="http://www.plos.org/support/donate.php" title="Join PLoS: Show Your Support">Join PLoS</a></li>
          <li><a href="http://blogs.plos.org/" title="PLoS Blogs">PLoS Blogs</a></li>
 	  <li><a href="http://www.zazzle.com/plos?rf=238434112415374154" title="PLoS Store">PLoS Store</a></li>
	  <li><a href="http://currents.plos.org/" title="PLoS Currents">PLoS Currents</a></li>
          <li><a href="http://www.plos.org/connect.html" title="PLoS.org | Stay Connected">Stay Connected</a></li>
        </ul>
      </li>
      <li class="journalnav"><a href="http://hubs.plos.org/" tabindex="109">Hubs</a>
        <ul>
          <li><a href="http://hubs.plos.org/biodiversity" title="PLoS Hubs: Biodiversity">Biodiversity</a></li>
          <li><a href="http://clinicaltrials.ploshubs.org/" title="PLoS Hub for Clinical Trials">Clinical Trials</a></li>
        </ul>
    </li>
      <li class="journalnav"><a href="http://www.plosjournals.org/" tabindex="108">Journals</a>
        <ul>
	<li><a href="http://www.plosbiology.org/" title="PLoSBiology.org">PLoS Biology</a></li>
	<li><a href="http://www.plosmedicine.org/" title="PLoSMedicine.org">PLoS Medicine</a></li>
	<li><a href="http://www.ploscompbiol.org/" title="PLoSCompBiol.org">PLoS Computational Biology</a></li>
	<li><a href="http://www.plosgenetics.org/" title="PLoSGenetics.org">PLoS Genetics</a></li>
	<li><a href="http://www.plospathogens.org/" title="PLoSPathogens.org">PLoS Pathogens</a></li>
        <li><a href="http://www.plosone.org/" title="PLoSONE.org">PLoS ONE</a></li>
        <li><a href="http://www.plosntds.org/" title="PLoSNTDs.org">PLoS Neglected Tropical Diseases</a></li>
        </ul>
      </li>
    </ul>
  <!-- end : navigation -->
  </div>
  <!-- end : header --> 



<!-- begin : main content -->
<div id="content" class="article" style="visibility: visible;">
<!-- begin : right hand column -->

<div id="rhc" xpathlocation="noDialog">

  <div id="download" class="rhcBox_type1">
    <div class="wrap">
      <ul>
        <li class="download icon"><strong>Download:</strong>
          <a href="http://www.plosone.org/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0014758&representation=PDF" title="Download article PDF">PDF</a> |
          <a href="http://www.plosone.org/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0014758" title="Download citations">Citation</a> |
          <a href="http://www.plosone.org/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0014758&representation=XML" title="Download Article XML">XML</a>
        </li>
        <li class="print icon"><a href="#" onclick="window.print();return false;" title="Print Article"><strong>Print article</strong></a></li>
          <li class="reprint icon"><a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0014758&volume=&issue=&title=Effect%20of%20Xpcl1%20Activation%20and%20p27Kip1%20Loss%20on%20Gene%20Expression%20in%20Murine%20Lymphoma&author_name=Daniel%20A.%20Kuppers%2C%20Harry%20C.%20Hwang%2C%20Aimee%20L.%20Jackson%2C%20Peter%20S.%20Linsley%2C%20Bruce%20E.%20Clurman%2C%20Matthew%20L.%20Fero&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a> New & improved!</li>
      </ul>
    </div>
  </div>

  <div id="impact" class="rhcBox_type2">
    <h6>Metrics <a href="http://www.plosone.org/static/help.action#articleMetrics" class="replaced" id="info" title="More information"><span>info</span></a></h6>

    <div id="totalDataInRHC"></div>

    <div id="relatedCitesInRHC"></div>

    <div id="ratingRhc1">

            <h6>Average Rating <a href="http://www.plosone.org/rate/getArticleRatings.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0014758" class="rating">(0 User Ratings)</a></h6>
            <ol>
              <li>
                <ul class="star-rating rating inline-rating-rhc" title="overall">
                    <li class="current-rating overall-rating pct0">Currently 0/5 Stars.</li>
                </ul>
                <a href="javascript:void(0);" onclick="return ambra.domUtil.swapDisplayTextMode(this, 'ratingAverages', null, 'Hide all categories', 'See all categories');" class="rating catAvg">See all categories</a>
                <div id="ratingAverages">
                  <ol class="ratingaverageRatings.">
                    <li><label for="insight">Insight</label>
                      <ul class="star-rating rating" title="insight">
                        <li class="current-rating average pct0">Currently 0/5 Stars.</li>
                      </ul>
                    </li>
                    <li><label for="reliability">Reliability</label>
                      <ul class="star-rating rating" title="reliability">
                        <li class="current-rating average pct0">Currently 0/5 Stars.</li>
                      </ul>
                    </li>
                    <li><label for="style">Style</label>
                      <ul class="star-rating rating" title="style">
                        <li class="current-rating average pct0">Currently 0/5 Stars.</li>
                      </ul>
                    </li>
                  </ol>
                </div>
                <a href="http://www.plosone.org/user/secure/secureRedirect.action?goTo=%2Farticle%2Finfo%3Adoi%2F10.1371%2Fjournal.pone.0014758" class="rating">Rate This Article</a>
              </li>
            </ol>
            
    </div>
    <div class="more clearfix"><a href="http://www.plosone.org/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0014758" title="More information">More</a></div>
  </div>

  <div id="related" class="rhcBox_type2">
    <h6>Related Content</h6>
    <dl class="related inlineList">
      <dt>Related Subject Categories</dt>
      <dd><a href="http://www.plosone.org/article/browse.action?catName=Cell+Biology" title="Browse the Cell Biology Category">Cell Biology</a></dd>, 
      <dd><a href="http://www.plosone.org/article/browse.action?catName=Molecular+Biology" title="Browse the Molecular Biology Category">Molecular Biology</a></dd>, 
      <dd><a href="http://www.plosone.org/article/browse.action?catName=Oncology" title="Browse the Oncology Category">Oncology</a></dd>, 
      <dd><a href="http://www.plosone.org/article/browse.action?catName=Computational+Biology" title="Browse the Computational Biology Category">Computational Biology</a></dd>
    </dl>

    <dl class="related">
      <dt>Related Articles on the Web</dt>
      <dd><a href="http://scholar.google.com/scholar?hl=en&lr=&q=related:http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0014758&btnG=Search">Google Scholar</a></dd>
      <dd id="pubMedRelatedLIInRHC" style="display: none;"><a id="pubMedRelatedURLInRHC">PubMed</a></dd>
    </dl>

    <div class="more clearfix"><a href="http://www.plosone.org/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0014758" title="More information">More</a></div>
  </div>

  <div id="share" class="rhcBox_type2">
    <h6>Share this Article <a href="http://www.plosone.org/static/help.action#socialBookmarkLinks" class="replaced" id="info" title="More information"><span>info</span></a></h6>
    <ul>
      <li class="bookmarklets">

          <a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0014758" target="_new"> <img src="16x16_su_solid.gif" alt="StumbleUpon" title="Add to StumbleUpon" border="0"></a>
          <script>function fbs_click() {u='http://dx.plos.org/10.1371/journal.pone.0014758';t='Effect%20of%20%3Citalic%3EXpcl1%3C%2Fitalic%3E%20Activation%20and%20%3Citalic%3Ep27%3Csup%3EKip1%3C%2Fsup%3E%3C%2Fitalic%3E%20Loss%20on%20Gene%20Expression%20in%20Murine%20Lymphoma';window.open('http://www.facebook.com/sharer.php?u='+encodeURIComponent(u)+'&t='+encodeURIComponent(t),'sharer','toolbar=0,status=0,width=626,height=436');return false;}</script><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0014758" onclick="return fbs_click()"><img src="facebook_share_icon.gif" alt="Facebook" title="Add to Facebook"></a>       <!-- for mor info, see http://www.facebook.com/share_partners.php -->
          <script type="text/javascript">

            function bookmark_in_connotea(u) {
                a=false; x=window; e=x.encodeURIComponent; d=document;
                w=open('http://www.connotea.org/addpopup?continue=confirm&uri='+e(u),
                    'add', 'width=600, height=400, scrollbars, resizable');
                void(x.setTimeout('w.focus()',200));
            }
          </script>
          <a style="cursor: pointer;" onclick='javascript:bookmark_in_connotea("http://dx.plos.org/10.1371/journal.pone.0014758");'><img src="icon_connotea_16x16.gif" alt="Connotea" title="Add to Connotea"></a>
          <a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0014758&title=Effect%20of%20%3Citalic%3EXpcl1%3C%2Fitalic%3E%20Activation%20and%20%3Citalic%3Ep27%3Csup%3EKip1%3C%2Fsup%3E%3C%2Fitalic%3E%20Loss%20on%20Gene%20Expression%20in%20Murine%20Lymphoma" target="_new"><img src="icon_citeulike_16x16.gif" alt="CiteULike" title="Add to CiteULike"></a>
          <a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0014758" title="Add to Mendeley" target="_new"><img src="icon_mendeley_16x16.gif" alt="Bibliography"></a>
          <script type="text/javascript">

            var twtTitle  = 'Effect of <italic>Xpcl1</italic> Activation and <italic>p27<sup>Kip1</sup></italic> Loss on Gene Expression in Murine Lymphoma';
            var twtUrl    = 'http://dx.plos.org/10.1371/journal.pone.0014758';
            var twtTag    = '#PLoS:'
            var maxLength = 140 - (twtUrl.length + 1);
            if (twtTitle.length > maxLength) {
            twtTitle = twtTitle.substr(0, (maxLength - 10))+'...';  //truncates title to include space for twtTag and ellipsis
            }
            var twtLink = 'http://twitter.com/home?status='+encodeURIComponent(twtTag + ' ' + twtTitle + ' ' + twtUrl);
            document.write('<a href="'+twtLink+'" target="_blank"'+'><img src="/images/icon_twitter.gif"  border="0" alt="Twitter icon" title="Tweet This!" /'+'><'+'/a>');
          </script>
          <script type="text/javascript">

          digg_url = 'http://dx.plos.org/10.1371/journal.pone.0014758';
          digg_skin = 'icon';
          digg_title = 'Effect of Xpcl1 Activation and p27Kip1 Loss on Gene Expression in Murine Lymphoma';
          digg_bodytext = '';
          digg_topic = '';
          digg_media = 'news';
          digg_window = 'new';

          </script>
          <script src="diggthis.js" type="text/javascript"></script>
      </li>
      <li class="email icon"><a href="http://www.plosone.org/article/emailArticle.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0014758" title="Email this article">Email this article</a></li>
    </ul>
  </div>
</div>
<!-- end : right hand column -->
<form name="articleInfo" id="articleInfo" method="" action="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0014758">
<input name="isAuthor" value="true" type="hidden">
<input name="authorIdList" value="" type="hidden">
<input name="userIdList" value="" type="hidden">
<input name="otherIdList" value="" type="hidden">
<input name="annotationId" value="" type="hidden">
<input name="isResearchArticle" value="true" type="hidden">
</form>
<div id="articleContainer"><span about="http://www.plos.org" typeof="foaf:Organization" property="foaf:name" style="display: none; visibility: hidden;">Public Library of Science</span>
<div id="researchArticle" class="content">
  <span rel="dc:publisher" href="http://www.plos.org"></span>
  <a id="top" name="top"></a>
<div id="contentHeader" xpathlocation="noSelect">
  <p>Open Access</p><p id="articleType">Research Article</p>
</div>

  <h1 xpathlocation="noSelect" property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">Effect of <i>Xpcl1</i> Activation and <i>p27<sup>Kip1</sup></i> Loss on Gene Expression in Murine Lymphoma</h1>
  <div property="dc:description" datatype="" style="display: none; visibility: hidden;">
         <p>Mice lacking the <i>p27<sup>Kip1</sup></i> Cdk inhibitor (<i>Cdkn1b</i>) exhibit increased susceptibility to lymphomas from the Maloney murine leukemia virus (M-MuLV), and exhibit a high frequency of viral integrations at <i>Xpcl1</i> (<i>Kis2</i>), a locus on the X-chromosome. <i>Xpcl1</i> encodes miR-106a&sim;363, a cluster of microRNAs that are expressed in response to adjacent retroviral integrations. We report the first large-scale profile of microRNA expression in MuLV-induced lymphomas, in combination with microarray gene expression analysis. The source material was T-cell lymphomas induced by M-MuLV in <i>p27<sup>Kip1</sup></i> knockout mice and normal thymus. Surprisingly, the overall levels of miRNA expression were equivalent in lymphomas and normal thymus. Nonetheless, the expression of specific microRNAs was altered in tumors. The miR-106a&sim;363 miRNA were over-expressed in lymphomas, particularly those with viral integrations at the <i>Xpcl1</i> locus. In contrast, <i>p27<sup>Kip1</sup></i> deletion itself was associated with a different pattern of microRNA expression. Gene expression was dramatically altered in lymphomas, yet paralleled data from T-cell lymphomas induced by other mechanisms. Genes with altered expression in association with the <i>p27<sup>Kip1</sup></i> null genotype were of similar functional classes to those associated with <i>Xpcl1</i> integration, but with the opposite pattern of expression. Thus, the effect of <i>p27<sup>Kip1</sup></i> deletion may be to oppose an anti-oncogenic effect of <i>Xpcl1</i> rather than enhancing its oncogenic functions. A subset of miR-106a&sim;363 target genes was consistently reduced in lymphomas with <i>Xpcl1</i> integrations, particularly genes with cell cycle and immune functions. We identify four predicted target genes of miR-106a&sim;363 miRNA, including <i>N-Myc (Mycn)</i>, and the TGF-beta receptor (<i>Tgfbr2</i>) using 3'UTR reporter assays. Still, bioinformatic miRNA target predictions were poor predictors of altered gene expression in lymphomas with <i>Xpcl1</i> integration. Confirmation of miR-106a&sim;363 gene targeting relevant to the tumor phenotype requires <i>in vivo</i> validation, because only a subset of predicted targets are consistently reduced in tumors that overexpress miR-106a&sim;363.</p>
      </div>
  <span property="dc:date" content="2011-03-11" datatype="xsd:date" rel="dc:identifier" href="http://dx.doi.org/10.1371/journal.pone.0014758"></span>
  <span property="dc:subject" content="Cell Biology"></span>
  <span property="dc:subject" content="Molecular Biology"></span>
  <span property="dc:subject" content="Gene Expression"></span>
  <span property="dc:subject" content="Hematological Malignancies"></span>
  <span property="dc:subject" content="Oncology"></span>
  <span property="dc:subject" content="Bioinformatics"></span>
  <span property="dc:subject" content="Computational Biology"></span>
  <span property="dc:subject" content="Genomics"></span>
<form action="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0014758">
  <input name="journalDisplayName" id="journalDisplayName" value="PLoS ONE" type="hidden">
  <input name="crossRefPageURL" id="crossRefPageURL" value="/article/crossref/info%3Adoi%2F10.1371%2Fjournal.pone.0014758" type="hidden">
  <input name="metricsTabURL" id="metricsTabURL" value="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0014758" type="hidden">
  <input name="doi" id="doi" value="info:doi/10.1371/journal.pone.0014758" type="hidden">
  <input name="articleTitleUnformatted" id="articleTitleUnformatted" value="Effect%20of%20Xpcl1%20Activation%20and%20p27Kip1%20Loss%20on%20Gene%20Expression%20in%20Murine%20Lymphoma" type="hidden">
  <input name="articlePubDate" id="articlePubDate" value="1299830400000" type="hidden">
</form>
<div class="horizontalTabs" xpathlocation="noSelect">
  <ul id="tabsContainer">
        <li id="article" class="active"><a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0014758" class="tab" title="Article">Article</a></li>
        <li id="metrics"><a href="http://www.plosone.org/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0014758" class="tab" title="Metrics">Metrics</a></li>
        <li id="related"><a href="http://www.plosone.org/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0014758" class="tab" title="Related Content">Related Content</a></li>
        <li id="comments"><a href="http://www.plosone.org/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0014758" class="tab" title="Comments">Comments: 0</a></li>
  </ul>
</div>
  <div id="articleMenu" xpathlocation="noSelect"> 
    <div class="wrap"> 
      <ul> 
        <li class="annotation icon">To <strong>add a note</strong>, highlight some text. <a href="#" onclick="toggleAnnotation(this, 'public'); return false;" title="Click to turn notes on/off">Hide notes</a></li> 
        <li class="discuss icon"> 
            <a href="http://www.plosone.org/user/secure/secureRedirect.action?goTo=%2Farticle%2Finfo%3Adoi%2F10.1371%2Fjournal.pone.0014758">Make a general comment</a> 
        </li> 
      </ul>
      <div id="sectionNavTopBox" style="display: none;">
        <p><strong>Jump to</strong></p> 
        <div id="sectionNavTop" class="tools"></div>
      </div>
    </div>
  </div>





<p class="authors" xpathlocation="noSelect"><span rel="dc:creator"><span property="foaf:name">Daniel A. Kuppers</span></span><sup><a href="#aff1">1</a></sup>, <span rel="dc:creator"><span property="foaf:name">Harry C. Hwang</span></span><sup><a href="#aff1">1</a></sup><sup><a href="#fn1">&curren;a</a></sup>, <span rel="dc:creator"><span property="foaf:name">Aimee L. Jackson</span></span><sup><a href="#aff2">2</a></sup><sup><a href="#fn2">&curren;b</a></sup>, <span rel="dc:creator"><span property="foaf:name">Peter S. Linsley</span></span><sup><a href="#aff2">2</a></sup>, <span rel="dc:creator"><span property="foaf:name">Bruce E. Clurman</span></span><sup><a href="#aff1">1</a></sup>, <span rel="dc:creator"><span property="foaf:name">Matthew L. Fero</span></span><sup><a href="#aff1">1</a></sup><sup><a href="#cor1" class="fnoteref">*</a></sup></p><p class="affiliations" xpathlocation="noSelect"><a name="aff1" id="aff1"></a><strong>1</strong> Clinical Research Division, Fred Hutchinson Cancer Research Center, Seattle, Washington, United States of America, <a name="aff2" id="aff2"></a><strong>2</strong> Rosetta Inpharmatics, Seattle, Washington, United States of America</p><div class="abstract" xpathlocation="/article[1]/front[1]/article-meta[1]/abstract[1]"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2 xpathlocation="noSelect">Abstract&nbsp;<a href="#top">Top</a></h2><p xpathlocation="/article[1]/front[1]/article-meta[1]/abstract[1]/p[1]">Mice lacking the <em>p27<sup>Kip1</sup></em> Cdk inhibitor (<em>Cdkn1b</em>) exhibit increased susceptibility to lymphomas from the Maloney murine leukemia virus (M-MuLV), and exhibit a high frequency of viral integrations at <em>Xpcl1</em> (<em>Kis2</em>), a locus on the X-chromosome. <em>Xpcl1</em> encodes miR-106a~363, a cluster of microRNAs that are expressed in response to adjacent retroviral integrations. We report the first large-scale profile of microRNA expression in MuLV-induced lymphomas, in combination with microarray gene expression analysis. The source material was T-cell lymphomas induced by M-MuLV in <em>p27<sup>Kip1</sup></em> knockout mice and normal thymus. Surprisingly, the overall levels of miRNA expression were equivalent in lymphomas and normal thymus. Nonetheless, the expression of specific microRNAs was altered in tumors. The miR-106a~363 miRNA were over-expressed in lymphomas, particularly those with viral integrations at the <em>Xpcl1</em> locus. In contrast, <em>p27<sup>Kip1</sup></em> deletion itself was associated with a different pattern of microRNA expression. Gene expression was dramatically altered in lymphomas, yet paralleled data from T-cell lymphomas induced by other mechanisms. Genes with altered expression in association with the <em>p27<sup>Kip1</sup></em> null genotype were of similar functional classes to those associated with <em>Xpcl1</em> integration, but with the opposite pattern of expression. Thus, the effect of <em>p27<sup>Kip1</sup></em> deletion may be to oppose an anti-oncogenic effect of <em>Xpcl1</em> rather than enhancing its oncogenic functions. A subset of miR-106a~363 target genes was consistently reduced in lymphomas with <em>Xpcl1</em> integrations, particularly genes with cell cycle and immune functions. We identify four predicted target genes of miR-106a~363 miRNA, including <em>N-Myc (Mycn)</em>, and the TGF-beta receptor (<em>Tgfbr2</em>) using 3'UTR reporter assays. Still, bioinformatic miRNA target predictions were poor predictors of altered gene expression in lymphomas with <em>Xpcl1</em> integration. Confirmation of miR-106a~363 gene targeting relevant to the tumor phenotype requires <em>in vivo</em> validation, because only a subset of predicted targets are consistently reduced in tumors that overexpress miR-106a~363.</p>
</div>


<div class="articleinfo" xpathlocation="noSelect"><p><strong>Citation: </strong>Kuppers DA, Hwang HC, Jackson AL, Linsley PS, Clurman BE, et al.  (2011) Effect of <em>Xpcl1</em> Activation and <em>p27<sup>Kip1</sup></em> Loss on Gene Expression in Murine Lymphoma. PLoS ONE 6(3):
          e14758.
            doi:10.1371/journal.pone.0014758</p><p><strong>Editor: </strong>Grzegorz Kudla, University of Edinburgh, United Kingdom</p><p></p><p><strong>Received:</strong> April 9, 2010; <strong>Accepted:</strong> February 14, 2011; <strong>Published:</strong> March 11, 2011</p><p><strong>Copyright:</strong> &copy; 2011 Kuppers et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding:</strong> National Cancer Institute R01CA100053 (MLF), R01CA084069 (BEC), K08HL073670 (HCH) <a href="http://www.cancer.gov/">http://www.cancer.gov/</a>, Rosetta Inpharmatics, Inc. (PSL, ALJ) provided materials and technical support for microarray processing and miRNA QPCR. The funders had no role in data analysis, the decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> Salary support in the form of research grants from the NIH and NCI was provided to M. Fero, H. Hwang, B. Clurman. Both Regulus Inc. and Rosetta Inpharmatics, Inc (a subsidiary of Merck Inc.) have approved or pending patents involving miRNA and microarray technology. The Fred Hutchinson Cancer Research Center holds patents on p27-deficient mouse models. All individuals received salary support from their host institutions. This does not alter the authors' adherence to all the PLoS ONE policies on sharing data and materials.</p><p><a name="cor1"></a>* E-mail: <a href="mailto:mfero@fhcrc.org">mfero@fhcrc.org</a></p><p><a name="fn1"></a>&curren;a<span class="capture-id"> Current address: PhenoPath Laboratories, Seattle, Washington, United States of America</span></p><p><a name="fn2"></a>&curren;b<span class="capture-id"> Current address: Regulus Therapeutics, San Diego, California, United States of America</span></p></div>





<div id="section1" xpathlocation="/article[1]/body[1]/sec[1]"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3 xpathlocation="noSelect">Introduction&nbsp;<a href="#top">Top</a></h3><p xpathlocation="/article[1]/body[1]/sec[1]/p[1]">Mice lacking one or both copies of the <em>p27<sup>Kip1</sup></em> Cdk inhibitor are susceptible to developing a variety of tumor types when exposed to mutagenic agents. <a href="#pone.0014758-Fero1">[1]</a>, <a href="#pone.0014758-Gao1">[2]</a> Several oncogenes and tumor suppressor genes have been shown to cooperate with <em>p27<sup>Kip1</sup></em> loss, including PTEN and APC, in the induction of breast, colon and prostate cancers in mouse models. <a href="#pone.0014758-Gao1">[2]</a>&ndash;<a href="#pone.0014758-Petrocelli1">[4]</a> We previously identified <em>Xpcl1</em> as a common viral integration site in T-cell lymphomas induced by Moloney murine leukemia virus (M-MuLV) infection. M-MuLV integrations targeted <em>Xpcl1</em> at a higher frequently in tumors induced in <em>p27<sup>Kip1</sup></em> knockout mice. This finding suggested that <em>p27<sup>Kip1</sup></em> deletion cooperates with <em>Xpcl1</em> activation in tumorigenesis, thus the region was referred to as the <em>X-chromosome p27 cooperating locus</em>. <a href="#pone.0014758-Hwang1">[5]</a> Expression of a non-coding RNA from this locus was increased in tumors with <em>Xpcl1</em> integrations, and this was higher than levels expressed in normal thymic tissue. <em>Xpcl1</em> is not exclusively targeted by M-MuLV in mice with <em>p27<sup>Kip1</sup></em> deletions; it has also been identified as a viral integration site in p27+/&minus; and wildtype animals. <a href="#pone.0014758-Hwang1">[5]</a>, <a href="#pone.0014758-GarrettEngele1">[6]</a> Likewise, integrations of other retroviruses, RadLV and SL3-3, have been independently reported to map to the same X-chromosomal site in murine lymphomas. <a href="#pone.0014758-Landais1">[7]</a>, <a href="#pone.0014758-Lum1">[8]</a> These integrations were associated with increased expression of a cluster of additional non-coding RNA splice variants, proximal to the viral integrations that were collectively referred to as <em>Kis2</em>. Mapping of microRNA sequences to the murine and human genomes identified the locus as the coding sequence for a cluster of five microRNAs, commonly referred to as miR-106a~363, which span the proximal and distal EST clusters. <a href="#pone.0014758-Tanzer1">[9]</a> Thus the normal gene, herein referred to as <em>Xpcl1</em>, apparently produces a primary transcript that is cleaved to form component pre-miRNA hairpins, as well as the flanking ESTs. Indeed, the individual microRNA comprising miR-106a~363 have been shown to be increased in tumors with viral integrations at the locus. <a href="#pone.0014758-Lum1">[8]</a>, <a href="#pone.0014758-Landais2">[10]</a> Large scale viral mutagenesis screens identified increased cointegration of retroviruses at <em>Xpcl1</em> and cyclin D3 loci, which further suggests an interaction between the Rb pathway with <em>Xpcl1</em> in tumorigenesis. <a href="#pone.0014758-Uren1">[11]</a></p>
<p xpathlocation="/article[1]/body[1]/sec[1]/p[2]">MicroRNAs are products of RNA precursors transcribed by RNA polymerase II and cleaved by the nuclear microprocessor complex, which is comprised of the RNAse III, Drosha, and DGCR8/Pasha, yielding 50&minus;80 bp stem-loop pre-miRNA (reviewed by R. Shivdasani). <a href="#pone.0014758-Shivdasani1">[12]</a> Cytoplasmic Dicer further cleaves the hairpin structures off pre-miRNA producing double stranded RNA that includes the mature 20&minus;22 bp microRNA product. Mature miRNA enter the RNA induced silencing complex (RISC), where recognition or target mRNA sequences may lead to translational silencing or degradation of mRNA targets. <a href="#pone.0014758-Ambros1">[13]</a> The miRBase database currently lists 600 microRNAs in the mouse genome. <a href="#pone.0014758-GriffithsJones1">[14]</a> MicroRNA have been shown to impact a wide variety of physiological activities including developmental timing, cell proliferation, cell death, and hematopoiesis through a combination of direct effects on target gene expression and a cascade of secondary effects. <a href="#pone.0014758-Carthew1">[15]</a> The first oncogenic non-protein encoding RNA discovered was <em>BIC</em>, which lies at a common integration site of the avian leukosis virus, and encodes miR-155. <a href="#pone.0014758-Clurman1">[16]</a>, <a href="#pone.0014758-Tam1">[17]</a> MiR-155 is overexpressed in a variety of human B-cell lymphomas, and <em>BIC</em> overexpression induced lymphomas in transgenic mice. <a href="#pone.0014758-Costinean1">[18]</a>&ndash; Human B-cell lymphomas have also been shown to harbor gene rearrangements involving the miR-17~92 microRNA cluster, which is a paralog of miR-106a~363. Overexpression of miR-17~92 in transgenic mice increased the rate of Myc-induced lymphomas and was associated with reduced apoptosis, whereas cells lacking miR-17~92 are prone to Myc induced apoptosis. <a href="#pone.0014758-Mu1">[21]</a>, <a href="#pone.0014758-He1">[22]</a></p>
<p xpathlocation="/article[1]/body[1]/sec[1]/p[3]">It has been reported that tumor cells exhibit a global reduction in miRNA expression compared to normal tissues, and that impairment of the miRNA processing machinery is oncogenic. <a href="#pone.0014758-Kumar1">[23]</a>, <a href="#pone.0014758-Lu1">[24]</a> Yet, little is known about the global patterns of miRNA or protein-coding gene expression in lymphomas induced by murine retroviruses. A hypothesis of the current work is that genetic alterations of M-MuLV induced lymphomas would result in characteristic phenotypic patterns of miRNA and protein-coding gene expression. Deletion of <em>p27<sup>Kip1</sup></em> has been shown to be associated with more rapid M-MuLV-induced lymphoma development, with a higher frequency of viral integrations at <em>Xpcl1</em>. We therefore hypothesized that the gene expression phenotype associated with <em>p27<sup>Kip1</sup></em> deletion in tumors would reflect an enhancement of the tumor phenotype, or an enhancement of the miRNA or gene expression profile specific to <em>Xpcl1</em>. The classes of genes associated with <em>p27<sup>Kip1</sup></em> loss or <em>Xpcl1</em> activation may give clues to the altered functions associated with these mutations. Herein, we report the results of both a miRNA and a protein-coding gene expression analysis from M-MuLV induced T-cell lymphomas. We also validate the targeting of four genes by miRNA from the miR-106a~363 cluster.</p>
</div>

<div id="section2" xpathlocation="/article[1]/body[1]/sec[2]"><a id="s2" name="s2" toc="s2" title="Results"></a><h3 xpathlocation="noSelect">Results&nbsp;<a href="#top">Top</a></h3>
<h4 xpathlocation="/article[1]/body[1]/sec[2]/sec[1]/title[1]">Global miRNA expression in lymphomas vs. normal thymus</h4>
<p xpathlocation="/article[1]/body[1]/sec[2]/sec[1]/p[1]">Global assessments of microRNA expression have not been reported in MuLV-induced lymphomas. In this study, we used a qPCR platform (previously described) to assess expression of 188 microRNAs. <a href="#pone.0014758-Jackson1">[25]</a> The source material was T-cell lymphomas that had been induced by the M-MuLV infection of mice, with or without <em>p27<sup>Kip1</sup></em> knockout mutations, in F1 hybrid (C57BL/6J &times; 129S4) mice, as well as normal thymus controls. Retroviral integration sites were previously mapped in these samples, and integrations at the <em>Xpcl1</em> locus were identified in tumors from <em>p27<sup>Kip1</sup></em> null animals, as well as in tumors arising in mice with one or more intact copies of <em>p27<sup>Kip1</sup></em>. <a href="#pone.0014758-Hwang1">[5]</a>, <a href="#pone.0014758-GarrettEngele1">[6]</a> (The complete dataset is given in <a href="#pone.0014758.s001">Table S1</a>) We first compared the global miRNA expression profile of all tumors in comparison with normal murine thymus. The abundance of specific microRNAs varied over a wide range (<a href="#pone-0014758-g001">Figure 1A</a>). In keeping with what has been reported in human cancers, we found that the majority of microRNAs were reduced in lymphomas relative to total RNA input. <a href="#pone.0014758-Lu1">[24]</a> However, this approach does not consider the fact that lymphoma cells have a higher total RNA content than does normal thymocytes. We therefore measured the RNA and DNA content in each tumor and compared this to normal thymic tissue. On average, the lymphomas contained 1.9-fold more RNA relative to tissue mass, and relative to genomic DNA mass. Considering that M-MuLV tumors are near diploid, we conclude that there was nearly a two-fold increase in total RNA per cell in the lymphomas. <a href="#pone.0014758-Vasmel1">[26]</a> When adjusting for the total RNA content per cell, we find that the number of miRNAs significantly increased in lymphomas is comparable to the number that are decreased (<a href="#pone-0014758-g001">Figure 1B</a>). Likewise, the sum total quantity of miRNA in lymphomas was equivalent to normal tissue (Fold change = 1.08&plusmn;0.05, tumor vs. normal, mean &plusmn; SE) when considered on a per cell basis.</p>
<div class="figure" xpathlocation="/article[1]/body[1]/sec[2]/sec[1]/fig[1]"><a name="pone-0014758-g001" id="pone-0014758-g001" title="Click for larger image " href="http://www.plosone.org/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0014758&imageURI=info:doi/10.1371/journal.pone.0014758.g001" onclick="window.open(this.href,'plosSlideshow','directories=no,location=no,menubar=no,resizable=yes,status=no,scrollbars=yes,toolbar=no,height=600,width=850');return false;"><img xpathlocation="noSelect" src="fetchobject.action" alt="thumbnail" class="thumbnail" align="left" border="1"></a><p><strong xpathlocation="/article[1]/body[1]/sec[2]/sec[1]/fig[1]/label[1]"><a xpathlocation="noSelect" href="http://www.plosone.org/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0014758&imageURI=info:doi/10.1371/journal.pone.0014758.g001" onclick="window.open(this.href,'plosSlideshow','directories=no,location=no,menubar=no,resizable=yes,status=no,scrollbars=yes,toolbar=no,height=600,width=850');return false;"><span>Figure 1. </span></a> <span xpathlocation="/article[1]/body[1]/sec[2]/sec[1]/fig[1]/caption[1]/title[1]">Expression of miRNA in M-MuLV lymphomas.</span></strong></p><p xpathlocation="/article[1]/body[1]/sec[2]/sec[1]/fig[1]/caption[1]/p[1]">The overall abundance and distribution of expression of individual miRNA species in (A) lymphomas, compared to normal thymus, is similar but varies over a wide range. Concentration is expressed as copies per 10 pg or input RNA. Of the 62 miRNA with expression levels&gt;250 copies/cell (B) the majority show increased expression in tumors. (Levels displayed as the fold change of copies per tumor cell relative to copies per normal thymocyte, on a log2 scale). All 5 miRNA from the miR-106a~363 cluster (green) are increased in tumors. The dashed red line shows magnitude of baseline shift resulting from a mean 1.9-fold increase in total RNA in tumors vs. normal thymus. Every miRNA value below this line is reduced relative to the increased total RNA content of tumor cells. Nine miRNA altered in association with <em>p27<sup>Kip1</sup></em> genotype are shown in a heatmap (C) reflecting the relative abundance of each miRNA across tumor samples. Heatmap of five miRNA (D) with changes associated with <em>Xpcl1</em> viral integration status. All 5 are increased and are members of the miR-106a~363 cluster.</p>
<span xpathlocation="noSelect">doi:10.1371/journal.pone.0014758.g001</span><div class="clearer"></div></div>

<h4 xpathlocation="/article[1]/body[1]/sec[2]/sec[2]/title[1]">miRNA associated with <em>p27<sup>Kip1</sup></em> null genotype and <em>Xpcl1</em> integrations</h4>
<p xpathlocation="/article[1]/body[1]/sec[2]/sec[2]/p[1]">Amongst tumor samples, we compared the global microRNA expression profiles associated with <em>p27<sup>Kip1</sup></em> genotype (<em>p27<sup>Kip1</sup></em> null vs. wildtype), as well as the status of viral integrations at <em>Xpcl1</em> (positive or negative). Unsupervised hierarchical clustering based on abundance of individual miRNA (not shown) did not strictly segregate tumor samples either by <em>p27<sup>Kip1</sup></em> genotype or by <em>Xpcl1</em> integration. This indicates that, for the majority of miRNA, neither of these factors are the major determinant of variability. Nonetheless, some of the most striking changes in miRNA expression in lymphomas included the microRNA encoded by the miR-106a-363 cluster (<a href="#pone-0014758-g001">Figure 1B</a>). Using a supervised analysis algorithm (PAM), we tested the hypothesis that viral integrations at <em>Xpcl1</em> are associated with expression of a distinct set of miRNA. The median false positive discovery rate (FDR) was zero at up to 5 microRNAs but rose sharply at higher numbers. This indicates that there are only 5 microRNAs (miR-363, miR-92-2, miR-20b, miR-19b, and miR-106a) influenced by viral integration at <em>Xpcl1</em> (<a href="#pone-0014758-g001">Figure 1D</a>). Sequence of a sixth microRNA, miR-18b, also lies in the cluster but it was not increased in <em>Xpcl1</em>+ tumors relative to <em>Xpc1</em>- tumors. Thus, the increased miR-18 levels noted in tumors (<a href="#pone-0014758-g001">Figure 1B</a>), may be due to expression from paralogous miR-17~92 cluster. One potential mechanism of cooperation would be if <em>p27<sup>Kip1</sup></em> deletion increased <em>Xpcl1</em> expression, e.g. by increasing expression following retroviral gene insertions or by increasing the miRNA stability. Therefore, we asked whether homozygous deletion of <em>p27<sup>Kip1</sup></em> was associated with altered miRNA expression in tumor samples. A supervised analysis (<a href="#pone-0014758-g001">Figure 1C</a>) suggests that 9 microRNAs were associated with the <em>p27<sup>Kip1</sup></em> null genotype, with considerable variability (FDR 30%). However, this list of miRNAs does not include those from the miR-106a~363 cluster. Likewise, limiting the analysis to tumors without <em>Xpcl1</em> integrations, also does not identify miR-106a~363 microRNA associated with the absence of <em>p27<sup>Kip1</sup></em> (not shown). Thus, <em>p27<sup>Kip1</sup></em> deletion does not appear to augment miR-106a~363 expression in lymphomas, but it may induce increases in other miRNAs.</p>


<h4 xpathlocation="/article[1]/body[1]/sec[2]/sec[3]/title[1]">Gene expression in M-MuLV induced tumors</h4>
<p xpathlocation="/article[1]/body[1]/sec[2]/sec[3]/p[1]">Because gene expression profiles have not been reported for M-MuLV induced lymphomas, we compared the general pattern of gene expression in tumors compared to normal thymic tissue. Our analysis normalized expression values within samples, as a function of both expression intensities, and 2-dimensional feature locations. <a href="#pone.0014758-Wang1">[27]</a> A supervised analysis (PAM) was used to identify genes differentially expressed in tumors (<a href="#pone-0014758-g002">Figure 2</a>) compared to normal thymus. In addition, tumors showed more heterogeneity of expression compared to normal tissue. In order to determine whether the pattern of expression seen in M-MuLV tumors was similar to other T-cell lymphomas we compared our data to that previously reported for T-cell lymphomas arising in mice with a &szlig;-catenin mutation. <a href="#pone.0014758-Guo1">[28]</a> We observed a high degree of correlation across the two data sets, which is shown in <a href="#pone-0014758-g002">Figure 2B</a>, a plot of expression rankings. The correlation is readily apparent as an increased density of points in the right upper and left lower quadrants (R<sup>2</sup> = 0.45, p&lt;2&times;10<sup>&minus;16</sup> Pearson correlation). In a gene ontology assessment, the biological function terms for DNA and protein metabolism and the cell division cycle, were enriched amongst genes with high levels of expression in tumors on either platform. In contrast, T-cell function and intracellular signaling terms were enriched in the genes with reduced expression in lymphomas (<a href="#pone.0014758.s002">Table S2</a>). We refer to this altered pattern of gene expression as the lymphoma expression phenotype.</p>
<div class="figure" xpathlocation="/article[1]/body[1]/sec[2]/sec[3]/fig[1]"><a name="pone-0014758-g002" id="pone-0014758-g002" title="Click for larger image " href="http://www.plosone.org/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0014758&imageURI=info:doi/10.1371/journal.pone.0014758.g002" onclick="window.open(this.href,'plosSlideshow','directories=no,location=no,menubar=no,resizable=yes,status=no,scrollbars=yes,toolbar=no,height=600,width=850');return false;"><img xpathlocation="noSelect" src="fetchobject_002.action" alt="thumbnail" class="thumbnail" align="left" border="1"></a><p><strong xpathlocation="/article[1]/body[1]/sec[2]/sec[3]/fig[1]/label[1]"><a xpathlocation="noSelect" href="http://www.plosone.org/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0014758&imageURI=info:doi/10.1371/journal.pone.0014758.g002" onclick="window.open(this.href,'plosSlideshow','directories=no,location=no,menubar=no,resizable=yes,status=no,scrollbars=yes,toolbar=no,height=600,width=850');return false;"><span>Figure 2. </span></a> <span xpathlocation="/article[1]/body[1]/sec[2]/sec[3]/fig[1]/caption[1]/title[1]">Patterns of mRNA expression in M-MuLV induced lymphomas.</span></strong></p><p xpathlocation="/article[1]/body[1]/sec[2]/sec[3]/fig[1]/caption[1]/p[1]">A heatmap (A) of the mRNA expression in 22 samples, showing both lymphomas (right) compared to normal thymus (left). The top 5000 genes (in terms of statistical significance) that display altered expression in tumors compared to normal thymus are shown. Expression levels are relative to gene-specific means, and are displayed log2 scale of microarray spot intensities (green  =  decreased, red  =  increased). A strong association of gene expression (B) in M-MuLV-induced lymphomas and &szlig;-catenin-induced lymphomas is seen in a plot of expression rankings. The 9804 genes with the highest levels of expression are shown color coded by plot density (left lower quadrant  =  reduced gene expression in tumors on both platforms, right upper quadrant  =  increased expression). An even distribution of points across all four quadrants would be expected if there is no association in tumor gene expression between the two tumor types. A heatmap of 250 genes (C) whose expression is most strongly associated with tumors containing <em>Xpcl1</em> viral integration (+) vs. tumors without <em>Xpcl1</em> integration (&minus;). Gene names are given for selected <em>Xpcl1</em> predicted target genes.</p>
<span xpathlocation="noSelect">doi:10.1371/journal.pone.0014758.g002</span><div class="clearer"></div></div>

<h4 xpathlocation="/article[1]/body[1]/sec[2]/sec[4]/title[1]">Gene expression profiles of <em>p27<sup>Kip1</sup></em> null and <em>Xpcl1</em>+ tumors</h4>
<p xpathlocation="/article[1]/body[1]/sec[2]/sec[4]/p[1]">To determine whether expression of specific classes of genes is associated with <em>p27<sup>Kip1</sup></em> genotype, we compared the microarray expression profiles of lymphomas from <em>p27<sup>Kip1</sup></em> null animals to tumors from mice with one or two copies of <em>p27<sup>Kip1</sup></em>. We ranked genes according to their PAM score, then tested for enrichment of gene ontology terms for the top 10% of genes altered in <em>p27<sup>Kip1</sup></em> null tumors, using the Panther gene classification system. <a href="#pone.0014758-Thomas1">[29]</a> In this analysis, statistical significance is a function of both the relative enrichment of classification terms and the size of the gene class, so we displayed results in circle plots (<a href="#pone-0014758-g003">Figure 3A&ndash;C</a>) showing all three parameters. Interestingly, the classes of genes enriched in <em>p27<sup>Kip1</sup></em> null tumors were similar to those enriched in tumors in general (<a href="#pone-0014758-g003">Figure 3A,B</a> and <a href="#pone.0014758.s002">Table S2</a>), suggesting that the <em>p27<sup>Kip1</sup></em> null tumors had a more pronounced lymphoma expression phenotype. In a separate analysis, we compared the patterns of gene expression in tumors with <em>Xpcl1</em> integrations vs. tumors without <em>Xpcl1</em> integrations, using an identical approach. There was partial overlap between <em>Xpcl1</em> and <em>p27<sup>Kip1</sup></em> genotypes, with 5 out of 8 <em>Xpcl1</em>+ tumors lacking <em>p27<sup>Kip1</sup></em>, and 4 out of 9 <em>Xpcl1</em>- tumors lacking <em>p27<sup>Kip1</sup></em>. For this reason, a slight resemblance between the classes of genes associated with <em>p27<sup>Kip1</sup></em> loss and <em>Xpcl1</em> integration might be expected. To the contrary, the functional classes associated with <em>p27<sup>Kip1</sup></em> null tumors showed the opposite pattern of expression in <em>Xpcl1</em>+ tumors. For example, the cell cycle term was enriched amongst genes with increased expression in <em>p27<sup>Kip1</sup></em> null tumors, but it was also enriched amongst genes decreased in <em>Xpcl1+</em> tumors (<a href="#pone-0014758-g003">Figure 3B,C</a> and <a href="#pone.0014758.s002">Table S2</a>). The opposite was true for genes in the Immunity and Defense category. These differences were largely due to changes in non-overlapping genes, as highlighted in the Venn diagram (<a href="#pone-0014758-g003">Figure 3D</a>), which further suggests that this observation is not simply due to the overlapping sample groups. To confirm the accuracy of the array results we verified, by qPCR, the RNA levels of 14 genes associated with <em>Xpcl1</em> integrations. There was a strong correlation (R<sup>2</sup> = 0.91, p = 5&times;10<sup>&minus;6</sup>, Pearson correlation) between mean expression levels determined by the microarrays and the qPCR assays (<a href="#pone-0014758-g004">Figure 4A</a>).</p>
<div class="figure" xpathlocation="/article[1]/body[1]/sec[2]/sec[4]/fig[1]"><a name="pone-0014758-g003" id="pone-0014758-g003" title="Click for larger image " href="http://www.plosone.org/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0014758&imageURI=info:doi/10.1371/journal.pone.0014758.g003" onclick="window.open(this.href,'plosSlideshow','directories=no,location=no,menubar=no,resizable=yes,status=no,scrollbars=yes,toolbar=no,height=600,width=850');return false;"><img xpathlocation="noSelect" src="fetchobject_003.action" alt="thumbnail" class="thumbnail" align="left" border="1"></a><p><strong xpathlocation="/article[1]/body[1]/sec[2]/sec[4]/fig[1]/label[1]"><a xpathlocation="noSelect" href="http://www.plosone.org/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0014758&imageURI=info:doi/10.1371/journal.pone.0014758.g003" onclick="window.open(this.href,'plosSlideshow','directories=no,location=no,menubar=no,resizable=yes,status=no,scrollbars=yes,toolbar=no,height=600,width=850');return false;"><span>Figure 3. </span></a> <span xpathlocation="/article[1]/body[1]/sec[2]/sec[4]/fig[1]/caption[1]/title[1]">Gene expression associated with <em>Xpcl1</em> integration and <em>p27<sup>Kip1</sup></em> null genotype.</span></strong></p><p xpathlocation="/article[1]/body[1]/sec[2]/sec[4]/fig[1]/caption[1]/p[1]">Biological function classes of the top 2500 genes associated with M-MuLV tumors vs. normal thymus (A) are shown in a circle plot with statistical significance plotted on the Y-axis and relative enrichment (observed/expected) plotted on the X-axis. Genes showing increased expression are shown (red) to the right and those with decreased expression (green) are plotted left of the Y-axis (P-values, binomial test). Circle diameters are proportional to the number of genes observed in each group. Within tumor samples, the functional classification of genes associated with (B) <em>p27<sup>Kip1</sup></em> null genotype or (C) <em>Xpcl1</em> integrations are shown. A Venn diagram (D) shows limited overlap amongst genes exhibiting decreased expression in association with <em>Xpcl1</em> integration (<em>Xpcl1</em> Low) and simultaneously showing increased expression in <em>p27<sup>Kip1</sup></em> null tumors (KO High) (n = 30). Likewise, there is little overlap between the <em>Xpcl1</em> High vs. KO Low sets (n = 37).</p>
<span xpathlocation="noSelect">doi:10.1371/journal.pone.0014758.g003</span><div class="clearer"></div></div><div class="figure" xpathlocation="/article[1]/body[1]/sec[2]/sec[4]/fig[2]"><a name="pone-0014758-g004" id="pone-0014758-g004" title="Click for larger image " href="http://www.plosone.org/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0014758&imageURI=info:doi/10.1371/journal.pone.0014758.g004" onclick="window.open(this.href,'plosSlideshow','directories=no,location=no,menubar=no,resizable=yes,status=no,scrollbars=yes,toolbar=no,height=600,width=850');return false;"><img xpathlocation="noSelect" src="fetchobject_004.action" alt="thumbnail" class="thumbnail" align="left" border="1"></a><p><strong xpathlocation="/article[1]/body[1]/sec[2]/sec[4]/fig[2]/label[1]"><a xpathlocation="noSelect" href="http://www.plosone.org/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0014758&imageURI=info:doi/10.1371/journal.pone.0014758.g004" onclick="window.open(this.href,'plosSlideshow','directories=no,location=no,menubar=no,resizable=yes,status=no,scrollbars=yes,toolbar=no,height=600,width=850');return false;"><span>Figure 4. </span></a> <span xpathlocation="/article[1]/body[1]/sec[2]/sec[4]/fig[2]/caption[1]/title[1]">Effect of <em>Xpcl1</em> on target gene expression.</span></strong></p><p xpathlocation="/article[1]/body[1]/sec[2]/sec[4]/fig[2]/caption[1]/p[1]">(A) Differences in mRNA expression was measured by qPCR (mean -&Delta;&Delta;Ct) for 17 genes and plotted as the mean difference in PCR cycle number of <em>Xpcl1</em> positive vs. <em>Xpcl1</em> negative tumors. This correlated with the changes in genes expression noted on microarrays (mean M, log2(tumor/normal)). Genes with varying levels of change in association with <em>Xpcl1</em> integration are shown, some of which are predicted targets of miR-106a~363 (red) and others which are not (blue). Comparison of luciferase activity (B) in reporter constructs containing 3'UTR sequences of miR-106a~363 predicted target genes: <em>Nfat5</em>, <em>N-myc</em>, and <em>Grsf1</em>. Each 3'UTR reporter construct was cotransfected with either pCIG-<em>Xpcl1</em> or empty pCIG vector. <em>Xpcl1</em> was also transfected with reporter constructs with mutations in the seed region of the predicted miRNA target site (&Delta;3'UTR). <em>Xpcl1</em> expression reduces luciferase expression associated with the 3'UTR of <em>Tgfbr2</em> (C), and results in reduced <em>Tgfbr2</em> mRNA levels as measured by qPCR. <em>Xpcl1</em> reduces downstream signaling in response to TGF-beta (right panel C) as measured by activity of the 3T3-Lux reporter construct. (Barplots are means &plusmn; SE).</p>
<span xpathlocation="noSelect">doi:10.1371/journal.pone.0014758.g004</span><div class="clearer"></div></div>

<h4 xpathlocation="/article[1]/body[1]/sec[2]/sec[5]/title[1]"><em>Xpcl1</em> target gene expression</h4>
<p xpathlocation="/article[1]/body[1]/sec[2]/sec[5]/p[1]">To test the hypothesis that miRNA expression was associated with reduced expression of predicted target genes we analyzed expression of genes that are predicted targets of microRNAs overexpressed in tumors with <em>Xpcl1</em> locus integrations, and microRNAs associated with <em>p27<sup>Kip1</sup></em> genotype. Within tumors, a number of miR-106a~363 target genes displayed reduced expression in association with <em>Xpcl1</em> integration. Of 500 <em>Xpcl1</em> predicted targets decreased in tumors with <em>Xpcl1</em> viral integrations there was relative enrichment for the terms Oncogenesis, and Cell Cycle (<a href="#pone-0014758-t001">Table 1</a>). We cloned the 3'UTR of four predicted target genes (<a href="#pone-0014758-g004">Figure 4B,C</a>), which showed varying levels of reduced expression in tumors, into a luciferase reporter constructs. This included two genes (<em>N-myc</em> and <em>Grsf1</em>) that were reduced in association with <em>Xpcl1</em> viral integration, and two genes (<em>Tgfbr2</em> and <em>Nfat5</em>) that were not. In each case, cotransfection of a genomic fragment of <em>Xpcl1</em> reduced expression of the 3'UTR reporter constructs. Mutation of miRNA binding site seed regions in the 3'UTR eliminated the effect of miR-106a~363 on the expression of <em>Nfat5</em> and <em>N-myc</em>, but did not restore <em>Grsf1</em> expression. In the case of <em>Tgfbr2</em>, miR-106a~363 decreased the expression level of the endogenous <em>Tgfbr2</em> mRNA and decreased TGF-beta signaling as measured by the p3TP-Lux reporter construct (<a href="#pone-0014758-g004">Figure 4D</a>). Still, the majority of miR-106a~363 predicted targets did not show a significant reduction at the RNA level in tumors with viral integration at <em>Xpcl1</em> (See <a href="#pone-0014758-g004">Figure 4A</a> and <a href="#pone.0014758.s003">Table S3</a>).</p>
<div class="figure" xpathlocation="/article[1]/body[1]/sec[2]/sec[5]/table-wrap[1]"><a name="pone-0014758-t001" id="pone-0014758-t001" title="Click for larger image " href="http://www.plosone.org/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0014758&imageURI=info:doi/10.1371/journal.pone.0014758.t001" onclick="window.open(this.href,'plosSlideshow','directories=no,location=no,menubar=no,resizable=yes,status=no,scrollbars=yes,toolbar=no,height=600,width=850');return false;"><img xpathlocation="noSelect" src="fetchobject_005.action" alt="thumbnail" class="thumbnail" align="left" border="1"></a><p><strong xpathlocation="/article[1]/body[1]/sec[2]/sec[5]/table-wrap[1]/label[1]"><a xpathlocation="noSelect" href="http://www.plosone.org/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0014758&imageURI=info:doi/10.1371/journal.pone.0014758.t001" onclick="window.open(this.href,'plosSlideshow','directories=no,location=no,menubar=no,resizable=yes,status=no,scrollbars=yes,toolbar=no,height=600,width=850');return false;"><span>Table 1. </span></a> <span xpathlocation="/article[1]/body[1]/sec[2]/sec[5]/table-wrap[1]/caption[1]/title[1]">Biological Process of <em>Xpcl1</em> targets with reduced mRNA expression.</span></strong></p><span xpathlocation="noSelect">doi:10.1371/journal.pone.0014758.t001</span><div class="clearer"></div></div>
</div>

<div id="section3" xpathlocation="/article[1]/body[1]/sec[3]"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3 xpathlocation="noSelect">Discussion&nbsp;<a href="#top">Top</a></h3><p xpathlocation="/article[1]/body[1]/sec[3]/p[1]">A principal aim of this study was to determine the pattern of miRNA and protein coding gene expression associated with <em>p27<sup>Kip1</sup></em> deletion or <em>Xpcl1</em> activation in M-MuLV lymphomas. Our initial hypothesis was that the two events would be associated with the activation of genes with similar functions, or in activities complementary for tumor development. Despite the extensive use of retroviral mutagenesis in mouse models, neither miRNA nor gene expression profiles have been previously reported. Thus, we felt it was imperative to observe the general patterns of gene expression in M-MuLV induced tumors compared to normal tissue, and determine whether this profile is comparable to T-cell lymphomas induced by other mechanisms. We found a global reduction in the miRNA/ total RNA ratio in tumors compared to normal thymic tissue, which is analogous to what has been described in human cancer. <a href="#pone.0014758-Lu1">[24]</a> However, it has long been recognized that proliferating cells, in general, and cancer cells, in particular, express higher levels of total RNA. <a href="#pone.0014758-Mellors1">[30]</a>, <a href="#pone.0014758-VonBertalanffy1">[31]</a> We therefore quantified the total RNA abundance in our M-MuLV lymphoma specimens, and found that the apparent change in global miRNA levels was entirely attributable to the increased total cellular RNA in tumors. Thus, it should not be assumed that reduced miRNA levels are a general characteristic of tumors, if the data is not adjusted to cell number. Still, in terms of biological impact, a decrease in the ratio of miRNA/total RNA may have the same implications as a drop in absolute miRNA abundance. The stoichiometry of miRNA compared to the expression of their target genes is likely a key determinant of miRNA effectiveness. In contrast to the lack of change of global miRNA levels, the pattern of individual microRNA expression was highly distinctive in tumors. A large numbers of microRNA were significantly changed (both up and down) compared to normal thymus.</p>
<p xpathlocation="/article[1]/body[1]/sec[3]/p[2]">We found that, of the 188 miRNA queried, the only miRNAs significantly altered in association with <em>Xpcl1</em> are members of the miR-106a~363 cluster. This reinforces observations reported for the radiation leukemia virus and SL3-3, which were also shown to be associated with increased miR-106a~363 miRNA expression. <a href="#pone.0014758-Lum1">[8]</a>, <a href="#pone.0014758-Landais2">[10]</a> Not all of the miR-106a~363 miRNA sequences are unique. However, the fact that we see overexpression of miR-363 (which is unique to this cluster) shows that the cluster is overexpressed. Mir-18, in contrast, is also encoded by the paralogous <em>miR-17~92</em> cluster. We observed increased expression of miR-18 in lymphomas, compared to normal thymic tissue. However, there was no further increase in miR-18 levels in tumors with <em>Xpcl1</em> integrations, so this likely represents expression from the <em>miR-17~92</em> paralog. The broad array of miRNAs sampled here indicates the specificity of the effect of Xpc1 integration on miR-106a~363, and argues against a general effect on miRNA expression. Deletion of <em>p27<sup>Kip1</sup></em> did not independently increase the expression of miR-106a~363, but instead may have influenced expression of a separate group of miRNAs. This argues against the hypothesis that p27Kip1 cooperates with <em>Xpcl1</em> simply by enhancing the expression of its constituent miRNAs.</p>
<p xpathlocation="/article[1]/body[1]/sec[3]/p[3]">In the case of protein coding genes, we observed a massive shift in the pattern of gene expression in M-MuLV induced T-cell lymphomas compared to normal thymic tissue. These changes are impressive when considered either on absolute terms (RNA per cell) or on relative terms (when adjusted to RNA abundance). On absolute terms, the overall abundance of RNA in tumors was 1.9&times; higher than in normal thymic tissue. Typically, expression data is normalized in ways that sets median gene expression values to a set value or, as in our case, sets the log ratios (sample vs. reference) to zero. Based on our raw (pre-normalized) data, 94% of all Entrez genes show significant increases in expression in tumors compared to normal thymic tissue (data not shown). Although this figure conveys the magnitude of shift in global RNA expression in tumor cells, it belies the shift in the pattern of gene expression exhibited by the tumors. In contrast, when expression data is subjected to conventional normalization (disregarding differences in RNA content per cell) we see that many genes exhibit relative decreases as well as increases in expression (<a href="#pone-0014758-g002">Figure 2A</a>). The functional classification of genes showing reduced expression in the lymphomas was frequently related to T-cell immunity (<a href="#pone-0014758-g003">Figure 3A</a>, <a href="#pone.0014758.s002">Table S2</a>). This is consistent with lymphomas developing a de-differentiated phenotype. Alternatively, it may reflect a reduced likelihood of up-regulating genes not essential to tumor development. Despite differences in array platforms, we observed a high level of similarity in gene expression in our M-MuLV lymphomas compared to that previously reported for lymphomas with a stabilizing mutation of &szlig;-catenin. <a href="#pone.0014758-Guo1">[28]</a> In both cases, genes involved with RNA and protein metabolism were up-regulated; whereas genes involved with immunity showed relatively reduced expression. We conclude that the massive alteration in gene expression in M-MuLV induced tumors is a common feature of murine T-cell lymphomas and is not a peculiarity of M-MuLV tumor induction.</p>
<p xpathlocation="/article[1]/body[1]/sec[3]/p[4]">In comparison to other tumors, gene expression in <em>p27<sup>Kip1</sup></em> null tumors showed further reduction in expression of genes involved with T-cell function (<a href="#pone-0014758-g003">Figure 3</a>). Thus, <em>p27<sup>Kip1</sup></em> loss was associated with both shortened survival and a further deviation from the normal T-cell expression phenotype. <a href="#pone.0014758-Hwang1">[5]</a> Interestingly, genes associated with <em>Xpcl1</em> retroviral integration showed the opposite pattern of expression, even though <em>Xpcl1</em> integration frequently occurred in <em>p27<sup>Kip1</sup></em> null animals. Our finding of a contrasting pattern of expression associated with <em>Xpcl1</em> genotype compared to <em>p27<sup>Kip1</sup></em> genotype does not support the hypothesis that loss of <em>p27<sup>Kip1</sup></em> simply enhances <em>Xpcl1</em>'s expression phenotype. Because of sample number limitations, our assessment did not include a pair-wise comparison of all four subsets of tumor samples based on <em>p27<sup>Kip1</sup></em> and <em>Xpcl1</em> genotypes. Such an approach might better elucidate the extent to which coincident mutation of <em>p27<sup>Kip1</sup></em> modifies gene expression changes by <em>Xpcl1</em>. Still, our data suggests that deletion of <em>p27<sup>Kip1</sup></em> counters an anti-oncogenic pattern of gene expression induced by <em>Xpcl1,</em> rather than enhancing an oncogenic effect.</p>
<p xpathlocation="/article[1]/body[1]/sec[3]/p[5]">According to the PicTar database 3,269 Entrez genes are predicted targets of at least one of the miR-106a~363 miRNAs. Considering the large number of predicted targets, it is expected that miR-106a~363 expression would induce a combination of oncogenic and anti-oncogenic effects. Others have demonstrated cellular activities by miR-106a~363 (and its paralogous miRNA clusters) that are potentially pro-oncogenic. This includes the demonstration that anchorage independent growth is enhanced by miR-106a~363 as well as direct targeting of <em>Mylip</em> and <em>Rbp1-like</em> gene expression. <a href="#pone.0014758-Landais2">[10]</a> Likewise, miR-106a~363 and its paralogs have been shown to promote proliferation of cultured cells by targeting p21<sup>Cip1/Waf1</sup> and p63. <a href="#pone.0014758-Ivanovska1">[32]</a>, <a href="#pone.0014758-Manni1">[33]</a> Thus it is somewhat surprising that our gene expression observations suggest that, in terms of its interaction with <em>Xpcl1</em>, the effect of <em>p27<sup>Kip1</sup></em> loss is to overcome an anti-tumor activity of <em>Xpcl1</em>, as opposed to enhancing <em>Xpcl1</em>'s oncogenic effect.</p>
<p xpathlocation="/article[1]/body[1]/sec[3]/p[6]">In regards to the expression of miRNA target genes, we saw a high frequency of reduced gene expression (52%) amongst genes that are predicted targets of miR-106a~363. However, we also observed a high frequency of reduced expression amongst miRNA target genes in general, regardless of the expression levels of their predicted effecter microRNAs. Thus, global reduction in miRNA target gene expression appears to be an incidental aspect of the tumor phenotype, although it might also result from secondary effects of a few overexpressed microRNAs. Two predicted targets of miR-106a~363 miRNA are <em>N-myc</em> and <em>Grsf1</em>, both of which were decreased in <em>Xpcl1</em> positive vs. <em>Xpcl1</em> negative tumors. M-MuLV integrations were present at the <em>N-Myc</em> locus in 3 of the <em>Xpcl1</em> negative tumors that we assayed (not shown). This may partially explain the reduced levels of <em>N-Myc</em> that we observed in <em>Xpcl1</em> positive tumors. Additionally, forced expression of <em>Xpcl1</em> in cultured T-cells reduced expression of an <em>N-Myc</em> 3'UTR reporter construct. This effect by <em>Xpcl1</em> was eliminated by mutation of the predicted miRNA target site in the <em>N-Myc</em> 3'UTR, which indicates that the miRNAs directly target <em>N-Myc</em>. Reduced <em>N-Myc</em> expression would be expected to have an anti-proliferative effect, in part because <em>N-Myc</em> activates transcription of the Skp2 ubiquitin ligase, which in turn degrades <em>p27Kip1</em> protein. <a href="#pone.0014758-Carrano1">[34]</a>, <a href="#pone.0014758-Muth1">[35]</a> Thus, the targeting of <em>N-Myc</em> by <em>Xpcl1</em> is one example of an anti-oncogenic effect that may be overcome by <em>p27Kip1</em> deletion.</p>
<p xpathlocation="/article[1]/body[1]/sec[3]/p[7]">Not all of the predicted targets of miR-106a~363 were consistently reduced in tumors with <em>Xpcl1</em> integrations. For example, <em>Tgfbr2</em>, the TGF-&szlig; cell surface receptor, is a predicted target of both miR-19b and miR-106a, but was not significantly changed in tumors with <em>Xpcl1</em> integration. Nonetheless, luciferase reporter assays indicate that the 3'UTR of <em>Tgfbr2</em> does confer sensitivity to <em>Xpcl1</em> expression (<a href="#pone-0014758-g004">Figure 4</a>). Likewise forced expression of <em>Xpcl1</em> in cultured cells reduced both <em>Tgfbr2</em> mRNA levels and reduced signaling downstream of TGF-&szlig;. To an extent, the discrepancy between tumor RNA expression and the reporter assay data may reflect the ability of the luciferase assay to detect the combined effect of miRNA on both mRNA stability and protein translation. However, endogenous gene expression is also subject to secondary effects and compensatory changes, which might easily overwhelm the comparatively subtle impact of miRNA. Finally, the existence of cell-type or gene-specific modifiers of miRNA function remains a possibility; so cultured T-cells may not accurately reflect the situation in tumors. Our list of miR-106a~363 predicted target genes, exhibiting reduced expression in association with <em>Xpcl1</em> integration, shortens the set of genes likely to mediate the effect of miR-106a~363 on the tumor phenotype. The confirmation of the oncogenic potential of <em>Xpcl1</em> and the elucidation of its pathogenic mechanism would further benefit from the <em>in vivo</em> manipulation of <em>Xpcl1</em> expression in the absence of other M-MuLV integrations. Given the complexity of gene expression in tumors, and the subtle yet widespread effects of miRNA expression, overexpression of <em>Xpcl1</em> in primary cells will likewise be important for the confirmation of physiologically relevant miR-106a~363 targets.</p>
</div>

<div id="section4" xpathlocation="/article[1]/body[1]/sec[4]"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3 xpathlocation="noSelect">Materials and Methods&nbsp;<a href="#top">Top</a></h3>
<h4 xpathlocation="/article[1]/body[1]/sec[4]/sec[1]/title[1]">Sample preparation and microRNA quantification</h4>
<p xpathlocation="/article[1]/body[1]/sec[4]/sec[1]/p[1]">This work involved frozen tumor samples previously taken from <em>p27<sup>Kip1</sup></em> knockout mice (p27&minus;/&minus;, p27+/&minus;, and p27+/+) induced by neonatal Maloney Murine Leukemia Virus (M-MuLV) infection. These tumors were previously surveyed for the genomic integration sites of M-MuLV proviruses and are thus characterized with respect to the presence or absence of M-MuLV integrations at the <em>Xpcl1</em> locus (<em>Xpcl1</em> positive), and Myc family members. For this work tumor tissue was characterized as being p27null (&minus;/&minus;) vs. not null (+/&minus; or +/+) and with respect to M-MuLV integration at the <em>Xpcl1</em> locus (positive or negative). <a href="#pone.0014758-Hwang1">[5]</a> The prior animal work was conducted according to national guidelines (OLAW) and approved by the IACUC of the Fred Hutchinson Cancer Research Center. Animals had been observed daily and were euthanized if they developed signs of morbidity, palpable tumors greater than 1 cm. diameter, or age &gt;1 year. These tumors were previously characterized for the presence of M-MuLV integrations at the <em>Xpcl1</em> locus. All animals were humanely euthanized with CO<sub>2</sub> or anesthetic inhalation. Frozen whole thymus from uninfected animals (on the same 129S4 &times; C57BL/6J F1 hybrid strain background) was used for comparisons. Tissues were divided, weighed and then extracted for either DNA or total RNA. RNA was prepared with Trizol (Invitrogen) using 1.5 volumes of isopropanol to increase the efficiency of miRNA precipitation. RNA and DNA were quantified using Ribogreen (Invitrogen) fluorescence. Expression of individual miRNAs were determined using a quantitative primer-extension PCR assay (<a href="#pone.0014758.s001">Table S1</a>), as previously described. <a href="#pone.0014758-Raymond1">[36]</a> Ct values were converted to copy numbers by interpolation on standard curves generated using single stranded mature miRNAs to calculate miRNA copies per 10 pg of input RNA, and are given in <a href="#pone.0014758.s001">Table S1</a>. The total RNA to DNA ratios of each tumor sample was determined by purifying total RNA and genomic DNA from weighed samples (&lt;5 mg) under non-saturating conditions on nucleic acid biding columns (RNeasy and DNeasy, QIAgen). Standard curves with 2-fold dilutions (0.625&ndash;10 mg) of starting material demonstrated a high degree of linearity with respect to the mass of starting material (R<sup>2</sup> = 0.985 for RNA, R<sup>2</sup> = 0.984 for DNA). The <em>pamr</em> software package for R (R-project.org, BioConductor.org) was used to identify miRNA associated with <em>Xpcl1</em> integration (<em>Xpcl1</em>+ vs. <em>Xpcl1</em>&minus;) and <em>p27<sup>Kip1</sup></em> genotype (null vs. +/&minus; or +/+). <a href="#pone.0014758-Tibshirani1">[37]</a> The top miRNAs associated with <em>p27<sup>Kip1</sup></em> genotype and <em>Xpcl1</em> integration are shown in heatmaps where quantities of individual miRNAs are displayed relative to the mean expression value of the same miRNAs across all samples. Hierarchical clustering was used to group samples post hoc using Cluster 3.0 software with Euclidean distance and complete linkage.</p>


<h4 xpathlocation="/article[1]/body[1]/sec[4]/sec[2]/title[1]">Microarray analysis</h4>
<p xpathlocation="/article[1]/body[1]/sec[4]/sec[2]/p[1]">Trizol prepped total RNA was additionally treated with DNAseI (RNAse free, Roche) and purified with RNeasy mini columns (Qiagen). RNA was amplified and labeled with Cy5 and Cy3 as previously described. <a href="#pone.0014758-Jackson1">[25]</a> Labeled RNA pooled from 17 tumor samples and 3 normal thymuses formed a common reference and were co-hybridized vs. sample RNA in a two color format on duplicate custom murine oligonucleotide arrays (Agilent, GEO accession GPL8525) (n = 40 arrays. See <a href="#pone.0014758.s005">Table S5</a> for complete sample descriptions). The genotypes of the tumor samples were balanced with roughly half of the samples (n = 9) being <em>p27<sup>Kip1</sup></em> null, and half of the samples containing <em>Xpcl1</em> viral integrations (n = 8), 4 of which were both <em>p27<sup>Kip1</sup></em> null and <em>Xpcl1</em>+. (See <a href="#pone.0014758.s005">5</a> for detailed sample descriptions). Log(2) ratios of fluorescence intensities (experimental sample vs. pooled reference) were normalized by intensity-dependent Loess followed by 2-dimensional location-dependent loess using the <em>marray</em> Bioconductor package without local background subtraction or scaling. <a href="#pone.0014758-Wang1">[27]</a> Missing (software flagged features) values were imputed by the K-nearest neighbor method, and genes associated with tumors vs. normal were determined with the PAM algorithm, using the <em>pamr</em> Bioconductor package. <a href="#pone.0014758-Troyanskaya1">[38]</a> We compared relative gene expression of tumors vs. normal, ranked by PAM-scores, to similarly processed raw data for &szlig;-catenin induced tumors obtained from the GEO repository. <a href="#pone.0014758-Guo1">[28]</a> Both studies included normal thymus so we used this as a common reference. Platform specific bias was avoided by comparing ranks of expression data (expressed as log tumor/normal) and calculating the geometric means across multiple samples. This approach is equivalent to the Rank Product (RP) method, which was shown to be robust for comparing gene expression profiles. <a href="#pone.0014758-Breitling1">[39]</a> Likewise, we analyzed our M-MuLV tumor data for genes associated with either <em>Xpcl1</em> integration or <em>p27<sup>Kip1</sup></em> genotype (PAM-scores listed in <a href="#pone.0014758.s003">Table S3</a>). Data processing was streamlined with custom wrapper functions contained in the <em>ArrayFun</em> package (available at <a href="http://labs.fhcrc.org/fero/R/ArrayFun.html">http://labs.fhcrc.org/fero/R/ArrayFun.html</a>). This package includes functions, (<em>array.getGEO, array.readGEO),</em> that downloads data from these 40 microarrays, along with the sample description table, from the GEO FTP site directly into R. Both MIAME compliant raw and normalized data are available at the GEO data repository (accession GSE16005). <a href="#pone.0014758-Edgar1">[40]</a> An R script that outlines for obtaining raw data and recapitulates the data processing steps is included as a supporting data file (<a href="#pone.0014758.s006">Table S6</a>). We used the &ldquo;compare gene list&rdquo; tool of the Panther Classification System (<a href="http://www.pantherdb.org/">http://www.pantherdb.org</a>) In order to determine the enrichment of biological function terms in genes of interest. <a href="#pone.0014758-Thomas1">[29]</a> We input the top 2500 Entrez Gene IDs (ranked by PAM scores, above) and compared this to the entire set genes on the arrays. Terms over-represented in the genes of interest (p-values&le;0.05, binomial test) are shown in <a href="#pone-0014758-g003">Figure 3</a> and are listed in <a href="#pone.0014758.s002">Table S2</a>.</p>


<h4 xpathlocation="/article[1]/body[1]/sec[4]/sec[3]/title[1]">MicroRNA target validation</h4>
<p xpathlocation="/article[1]/body[1]/sec[4]/sec[3]/p[1]">129S4 mouse genomic <em>Xpcl1</em> DNA was isolated from a phage library. A 1.5 kB fragment containing the entire miR-106a~363 miRNA cluster was subcloned into pCIG (courtesy of Andy McMahon) with the chicken B-actin promoter, the pSM30 vector (courtesy of Muneesh Tewari) and the p1026x vector with the Lck promoter (courtesy of Brian Iritani). <a href="#pone.0014758-Iritani1">[41]</a>&ndash;<a href="#pone.0014758-Wu1">[43]</a> Genomic 3'UTR DNA segments for putative target genes (<em>N-Myc</em>, <em>Tgfbr2</em>, <em>Grsf1</em>, and <em>Nfat5</em>) were PCR amplified from mouse 129S4 genomic DNA, cloned into pGL3-control (Promega), a luciferase reporter with the SV40 promoter, and verified by sequencing. Three or four bases of miRNA seed sequence, in the UTR of predicted target genes, were mutated using the QuickChange Mutagenesis Kit (Agilent, #210513) with oligonucleotides, listed in <a href="#pone.0014758.s004">Table S4</a>. Wildtype and mutant 3'UTR reporter constructs were co-transfected (in triplicate) with pSM30-<em>Xpcl1</em> vs. empty vector and pRL-TK (Promega, renilla control) into 293T cells using Lipofectamine 2000 (Invitrogen). <em>Tgfbr2</em> 3'UTR luciferase assays compared pCIG-Xpcl1 vs. empty vector. Luciferase activity was assayed at 48 hr. and adjusted by renilla activity. To quantify the effect on <em>Tgfbr2</em> mRNA expression and TGF-beta signaling, murine SV40-180 T-cells were transfected by electroporation with either p1026x&ndash;Xpcl1 or empty vector. RNA was isolated at 48 h and <em>Tgfbr2</em> qPCR was done, as described above. To measure TGF-beta signaling the p3TP-Lux reporter of TGF-beta signaling (courtesy of William Grady) was additionally transfected. The cells were treated with porcine TGF-beta 1 (5 ng/mL, R&D Systems) starting at 24 hrs. post-transfection and luciferase activity was assayed at 48 h. <a href="#pone.0014758-Wrana1">[44]</a> The transfection of each construct and its controls were performed in triplicate or quadruplicate.</p>

</div>

<div id="section5" xpathlocation="/article[1]/body[1]/sec[5]"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3 xpathlocation="noSelect">Supporting Information&nbsp;<a href="#top">Top</a></h3><a name="pone.0014758.s001" id="pone.0014758.s001"></a><p><strong xpathlocation="noSelect"><a href="http://www.plosone.org/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0014758.s001">Table S1. </a></strong></p><p xpathlocation="/article[1]/body[1]/sec[5]/supplementary-material[1]/caption[1]/p[1]">MicroRNA copy number per 10 pg input RNA.</p>
<p xpathlocation="/article[1]/body[1]/sec[5]/supplementary-material[1]/caption[1]/p[2]">(0.07 MB XLS)</p>
<a name="pone.0014758.s002" id="pone.0014758.s002"></a><p><strong xpathlocation="noSelect"><a href="http://www.plosone.org/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0014758.s002">Table S2. </a></strong></p><p xpathlocation="/article[1]/body[1]/sec[5]/supplementary-material[2]/caption[1]/p[1]">Biological Process of genes with altered expression in tumors.</p>
<p xpathlocation="/article[1]/body[1]/sec[5]/supplementary-material[2]/caption[1]/p[2]">(0.10 MB PDF)</p>
<a name="pone.0014758.s003" id="pone.0014758.s003"></a><p><strong xpathlocation="noSelect"><a href="http://www.plosone.org/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0014758.s003">Table S3. </a></strong></p><p xpathlocation="/article[1]/body[1]/sec[5]/supplementary-material[3]/caption[1]/p[1]">Expression array Gene IDs and PAM significance scores.</p>
<p xpathlocation="/article[1]/body[1]/sec[5]/supplementary-material[3]/caption[1]/p[2]">(0.96 MB XLS)</p>
<a name="pone.0014758.s004" id="pone.0014758.s004"></a><p><strong xpathlocation="noSelect"><a href="http://www.plosone.org/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0014758.s004">Table S4. </a></strong></p><p xpathlocation="/article[1]/body[1]/sec[5]/supplementary-material[4]/caption[1]/p[1]">Oligonucleotide sequences used for mutagenesis of the 3'UTR of miR- 106a~363 target genes.</p>
<p xpathlocation="/article[1]/body[1]/sec[5]/supplementary-material[4]/caption[1]/p[2]">(0.07 MB PDF)</p>
<a name="pone.0014758.s005" id="pone.0014758.s005"></a><p><strong xpathlocation="noSelect"><a href="http://www.plosone.org/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0014758.s005">Table S5. </a></strong></p><p xpathlocation="/article[1]/body[1]/sec[5]/supplementary-material[5]/caption[1]/p[1]">Descriptions of RNA samples used for expression arrays.</p>
<p xpathlocation="/article[1]/body[1]/sec[5]/supplementary-material[5]/caption[1]/p[2]">(0.02 MB XLS)</p>
<a name="pone.0014758.s006" id="pone.0014758.s006"></a><p><strong xpathlocation="noSelect"><a href="http://www.plosone.org/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0014758.s006">Table S6. </a></strong></p><p xpathlocation="/article[1]/body[1]/sec[5]/supplementary-material[6]/caption[1]/p[1]">Instructions and R code for acquisition and analysis of expression array data.</p>
<p xpathlocation="/article[1]/body[1]/sec[5]/supplementary-material[6]/caption[1]/p[2]">(0.00 MB TXT)</p>
</div>





<div xpathlocation="noSelect"><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3 xpathlocation="noSelect">Acknowledgments&nbsp;<a href="#top">Top</a></h3>
<p xpathlocation="/article[1]/back[1]/ack[1]/p[1]">We thank Jane Guo for assistance with sample preparation and Yanqun Wang at Merck, Inc. for assistance with raw data acquisition. We also thank William Grady and Muneesh Tewari for reporter plasmids.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3 xpathlocation="noSelect">Author Contributions&nbsp;<a href="#top">Top</a></h3><p xpathlocation="noSelect"><span class="capture-id">Conceived and designed the experiments: PSL MLF. Performed the experiments: DAK MLF. Analyzed the data: DAK ALJ MLF. Contributed reagents/materials/analysis tools: HCH ALJ PSL BEC MLF. Wrote the paper: MLF.</span></p></div><div xpathlocation="noSelect"><a id="references" name="references" toc="references" title="References"></a><h3 xpathlocation="noSelect">References&nbsp;<a href="#top">Top</a></h3><ol class="references" xpathlocation="noSelect"><li xpathlocation="noSelect"><a name="pone.0014758-Fero1" id="pone.0014758-Fero1"></a><span class="authors">Fero ML, Randel E, Gurley KE, Roberts JM, Kemp CJ</span> (1998) The murine gene p27Kip1 is haplo-insufficient for tumour suppression. Nature  396: 177&ndash;180. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Fero&title=The%20murine%20gene%20p27Kip1%20is%20haplo-insufficient%20for%20tumour%20suppression.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Gao1" id="pone.0014758-Gao1"></a><span class="authors">Gao H, Ouyang X, Banach-Petrosky W, Borowsky AD, Lin Y, et al. </span> (2004) A critical role for p27kip1 gene dosage in a mouse model of prostate carcinogenesis. Proc Natl Acad Sci USA  101: 17204&ndash;17209. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Gao&title=A%20critical%20role%20for%20p27kip1%20gene%20dosage%20in%20a%20mouse%20model%20of%20prostate%20carcinogenesis.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-DiCristofano1" id="pone.0014758-DiCristofano1"></a><span class="authors">Di Cristofano A, De Acetis M, Koff A, Cordon-Cardo C, Pandolfi PP</span> (2001) Pten and p27KIP1 cooperate in prostate cancer tumor suppression in the mouse. Nat Genet  27: 222&ndash;224. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Di%20Cristofano&title=Pten%20and%20p27KIP1%20cooperate%20in%20prostate%20cancer%20tumor%20suppression%20in%20the%20mouse.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Petrocelli1" id="pone.0014758-Petrocelli1"></a><span class="authors">Petrocelli T, Slingerland JM</span> (2001) PTEN deficiency: a role in mammary carcinogenesis. Breast Cancer Res  3: 356&ndash;360. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Petrocelli&title=PTEN%20deficiency:%20a%20role%20in%20mammary%20carcinogenesis.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Hwang1" id="pone.0014758-Hwang1"></a><span class="authors">Hwang HC, Martins CP, Bronkhorst Y, Randel E, Berns A, et al. </span> (2002) Identification of oncogenes collaborating with p27Kip1 loss by insertional mutagenesis and high-throughput insertion site analysis. Proc Natl Acad Sci U S A  99: 11293&ndash;11298. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Hwang&title=Identification%20of%20oncogenes%20collaborating%20with%20p27Kip1%20loss%20by%20insertional%20mutagenesis%20and%20high-throughput%20insertion%20site%20analysis.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-GarrettEngele1" id="pone.0014758-GarrettEngele1"></a><span class="authors">Garrett-Engele CM, Tasch MA, Hwang HC, Fero ML, Perlmutter RM, et al. </span> (2007) A mechanism misregulating p27 in tumors discovered in a functional genomic screen. PLoS Genet  3: e219. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Garrett-Engele&title=A%20mechanism%20misregulating%20p27%20in%20tumors%20discovered%20in%20a%20functional%20genomic%20screen.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Landais1" id="pone.0014758-Landais1"></a><span class="authors">Landais S, Quantin R, Rassart E</span> (2005) Radiation leukemia virus common integration at the Kis2 locus: simultaneous overexpression of a novel noncoding RNA and of the proximal Phf6 gene. J Virol  79: 11443&ndash;11456. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Landais&title=Radiation%20leukemia%20virus%20common%20integration%20at%20the%20Kis2%20locus:%20simultaneous%20overexpression%20of%20a%20novel%20noncoding%20RNA%20and%20of%20the%20proximal%20Phf6%20gene.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Lum1" id="pone.0014758-Lum1"></a><span class="authors">Lum AM, Wang BB, Li L, Channa N, Bartha G, Wabl M</span> (2007) Retroviral activation of the mir-106a microRNA cistron in T lymphoma. Retrovirology  4: 5. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Lum&title=Retroviral%20activation%20of%20the%20mir-106a%20microRNA%20cistron%20in%20T%20lymphoma.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Tanzer1" id="pone.0014758-Tanzer1"></a><span class="authors">Tanzer A, Stadler PF</span> (2004) Molecular evolution of a microRNA cluster. J Mol Biol  339: 327&ndash;335. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Tanzer&title=Molecular%20evolution%20of%20a%20microRNA%20cluster.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Landais2" id="pone.0014758-Landais2"></a><span class="authors">Landais S, Landry S, Legault P, Rassart E</span> (2007) Oncogenic potential of the miR-106-363 cluster and its implication in human T-cell leukemia. Cancer Res  67: 5699&ndash;5707. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Landais&title=Oncogenic%20potential%20of%20the%20miR-106-363%20cluster%20and%20its%20implication%20in%20human%20T-cell%20leukemia.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Uren1" id="pone.0014758-Uren1"></a><span class="authors">Uren AG, Kool J, Matentzoglu K, de Ridder J, Mattison J, et al. </span> (2008) Large-scale mutagenesis in p19(ARF)- and p53-deficient mice identifies cancer genes and their collaborative networks. Cell  133: 727&ndash;741. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Uren&title=Large-scale%20mutagenesis%20in%20p19%28ARF%29-%20and%20p53-deficient%20mice%20identifies%20cancer%20genes%20and%20their%20collaborative%20networks.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Shivdasani1" id="pone.0014758-Shivdasani1"></a><span class="authors">Shivdasani RA</span> (2006) MicroRNAs: regulators of gene expression and cell differentiation. Blood.  <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Shivdasani&title=MicroRNAs:%20regulators%20of%20gene%20expression%20and%20cell%20differentiation.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Ambros1" id="pone.0014758-Ambros1"></a><span class="authors">Ambros V</span> (2004) The functions of animal microRNAs. Nature  431: 350&ndash;355. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Ambros&title=The%20functions%20of%20animal%20microRNAs.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-GriffithsJones1" id="pone.0014758-GriffithsJones1"></a><span class="authors">Griffiths-Jones S, Saini HK, van Dongen S, Enright AJ</span> (2008) miRBase: tools for microRNA genomics. Nucleic Acids Res  36: D154&ndash;D158. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Griffiths-Jones&title=miRBase:%20tools%20for%20microRNA%20genomics.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Carthew1" id="pone.0014758-Carthew1"></a><span class="authors">Carthew RW, Sontheimer EJ</span> (2009) Origins and Mechanisms of miRNAs and siRNAs. Cell  136: 642&ndash;655. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Carthew&title=Origins%20and%20Mechanisms%20of%20miRNAs%20and%20siRNAs.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Clurman1" id="pone.0014758-Clurman1"></a><span class="authors">Clurman BE, Hayward WS</span> (1989) Multiple proto-oncogene activations in avian leukosis virus-induced lymphomas: evidence for stage-specific events. Mol Cell Biol  9: 2657&ndash;2664. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Clurman&title=Multiple%20proto-oncogene%20activations%20in%20avian%20leukosis%20virus-induced%20lymphomas:%20evidence%20for%20stage-specific%20events.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Tam1" id="pone.0014758-Tam1"></a><span class="authors">Tam W, Ben-Yehuda D, Hayward WS</span> (1997) bic, a novel gene activated by proviral insertions in avian leukosis virus-induced lymphomas, is likely to function through its noncoding RNA. Mol Cell Biol  17: 1490&ndash;1502. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Tam&title=bic,%20a%20novel%20gene%20activated%20by%20proviral%20insertions%20in%20avian%20leukosis%20virus-induced%20lymphomas,%20is%20likely%20to%20function%20through%20its%20noncoding%20RNA.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Costinean1" id="pone.0014758-Costinean1"></a><span class="authors">Costinean S, Zanesi N, Pekarsky Y, Tili E, Volinia S, et al. </span> (2006) Pre-B cell proliferation and lymphoblastic leukemia/high-grade lymphoma in E(mu)-miR155 transgenic mice. Proc Natl Acad Sci U S A  103: 7024&ndash;7029. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Costinean&title=Pre-B%20cell%20proliferation%20and%20lymphoblastic%20leukemia/high-grade%20lymphoma%20in%20E%28mu%29-miR155%20transgenic%20mice.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Eis1" id="pone.0014758-Eis1"></a><span class="authors">Eis PS, Tam W, Sun L, Chadburn A, Li Z, et al. </span> (2005) Accumulation of miR-155 and BIC RNA in human B cell lymphomas. Proc Natl Acad Sci U S A  102: 3627&ndash;3632. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Eis&title=Accumulation%20of%20miR-155%20and%20BIC%20RNA%20in%20human%20B%20cell%20lymphomas.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Kluiver1" id="pone.0014758-Kluiver1"></a><span class="authors">Kluiver J, Poppema S, de Jong D, Blokzijl T, Harms G, et al. </span> (2005) BIC and miR-155 are highly expressed in Hodgkin, primary mediastinal and diffuse large B cell lymphomas. J Pathol  207: 243&ndash;249. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Kluiver&title=BIC%20and%20miR-155%20are%20highly%20expressed%20in%20Hodgkin,%20primary%20mediastinal%20and%20diffuse%20large%20B%20cell%20lymphomas.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Mu1" id="pone.0014758-Mu1"></a><span class="authors">Mu P, Han YC, Betel D, Yao E, Squatrito M, et al. </span> (2009) Genetic dissection of the miR-17~92 cluster of microRNAs in Myc-induced B-cell lymphomas. Genes Dev  23: 2806&ndash;2811. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Mu&title=Genetic%20dissection%20of%20the%20miR-17%E2%88%BC92%20cluster%20of%20microRNAs%20in%20Myc-induced%20B-cell%20lymphomas.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-He1" id="pone.0014758-He1"></a><span class="authors">He L, Thomson JM, Hemann MT, Hernando-Monge E, Mu D, et al. </span> (2005) A microRNA polycistron as a potential human oncogene. Nature  435: 828&ndash;833. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=He&title=A%20microRNA%20polycistron%20as%20a%20potential%20human%20oncogene.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Kumar1" id="pone.0014758-Kumar1"></a><span class="authors">Kumar MS, Lu J, Mercer KL, Golub TR, Jacks T</span> (2007) Impaired microRNA processing enhances cellular transformation and tumorigenesis. Nat Genet  39: 673&ndash;677. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Kumar&title=Impaired%20microRNA%20processing%20enhances%20cellular%20transformation%20and%20tumorigenesis.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Lu1" id="pone.0014758-Lu1"></a><span class="authors">Lu J, Getz G, Miska EA, Alvarez-Saavedra E, Lamb J, et al. </span> (2005) MicroRNA expression profiles classify human cancers. Nature  435: 834&ndash;838. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Lu&title=MicroRNA%20expression%20profiles%20classify%20human%20cancers.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Jackson1" id="pone.0014758-Jackson1"></a><span class="authors">Jackson AL, Bartz SR, Schelter J, Kobayashi SV, Burchard J, et al. </span> (2003) Expression profiling reveals off-target gene regulation by RNAi. Nat Biotechnol  21: 635&ndash;637. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Jackson&title=Expression%20profiling%20reveals%20off-target%20gene%20regulation%20by%20RNAi.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Vasmel1" id="pone.0014758-Vasmel1"></a><span class="authors">Vasmel WL, Matthews EA, Gillis CP, Nieland J, Borst EA, et al. </span> (1989) Distinct chromosomal abnormalities in murine leukemia virus-induced T- and B-cell lymphomas. Int J Cancer  43: 1112&ndash;1119. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Vasmel&title=Distinct%20chromosomal%20abnormalities%20in%20murine%20leukemia%20virus-induced%20T-%20and%20B-cell%20lymphomas.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Wang1" id="pone.0014758-Wang1"></a><span class="authors">Wang J, Nygaard V, Smith-Sorensen B, Hovig E, Myklebost O</span> (2002) MArray: analysing single, replicated or reversed microarray experiments. Bioinformatics  18: 1139&ndash;1140. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Wang&title=MArray:%20analysing%20single,%20replicated%20or%20reversed%20microarray%20experiments.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Guo1" id="pone.0014758-Guo1"></a><span class="authors">Guo Z, Dose M, Kovalovsky D, Chang R, O'Neil J, et al. </span> (2007) Beta-catenin stabilization stalls the transition from double-positive to single-positive stage and predisposes thymocytes to malignant transformation. Blood  109: 5463&ndash;5472. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Guo&title=Beta-catenin%20stabilization%20stalls%20the%20transition%20from%20double-positive%20to%20single-positive%20stage%20and%20predisposes%20thymocytes%20to%20malignant%20transformation.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Thomas1" id="pone.0014758-Thomas1"></a><span class="authors">Thomas PD, Campbell MJ, Kejariwal A, Mi H, Karlak B, et al. </span> (2003) PANTHER: a library of protein families and subfamilies indexed by function. Genome Res  13: 2129&ndash;2141. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Thomas&title=PANTHER:%20a%20library%20of%20protein%20families%20and%20subfamilies%20indexed%20by%20function.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Mellors1" id="pone.0014758-Mellors1"></a><span class="authors">Mellors RC</span> (1956) Quantitative cytology and cytopathology: nucleic acids and proteins in the mitotic cycle of normal and neoplastic cells. Ann N Y Acad Sci  63: 1177&ndash;1201. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Mellors&title=Quantitative%20cytology%20and%20cytopathology:%20nucleic%20acids%20and%20proteins%20in%20the%20mitotic%20cycle%20of%20normal%20and%20neoplastic%20cells.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-VonBertalanffy1" id="pone.0014758-VonBertalanffy1"></a><span class="authors">Von Bertalanffy L, Masin M, Masin F, Kaplan L</span> (1957) Detection of gynecological cancer; use of fluorescence microscopy to show nucleic acids in malignant growth. Calif Med  87: 248&ndash;251. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Von%20Bertalanffy&title=Detection%20of%20gynecological%20cancer;%20use%20of%20fluorescence%20microscopy%20to%20show%20nucleic%20acids%20in%20malignant%20growth.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Ivanovska1" id="pone.0014758-Ivanovska1"></a><span class="authors">Ivanovska I, Ball AS, Diaz RL, Magnus JF, Kibukawa M, et al. </span> (2008) MicroRNAs in the miR-106b family regulate p21/CDKN1A and promote cell cycle progression. Mol Cell Biol  28: 2167&ndash;2174. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Ivanovska&title=MicroRNAs%20in%20the%20miR-106b%20family%20regulate%20p21/CDKN1A%20and%20promote%20cell%20cycle%20progression.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Manni1" id="pone.0014758-Manni1"></a><span class="authors">Manni I, Artuso S, Careccia S, Rizzo MG, Baserga R, et al. </span> (2009) The microRNA miR-92 increases proliferation of myeloid cells and by targeting p63 modulates the abundance of its isoforms. FASEB J  23: 3957&ndash;3966. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Manni&title=The%20microRNA%20miR-92%20increases%20proliferation%20of%20myeloid%20cells%20and%20by%20targeting%20p63%20modulates%20the%20abundance%20of%20its%20isoforms.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Carrano1" id="pone.0014758-Carrano1"></a><span class="authors">Carrano AC, Eytan E, Hershko A, Pagano M</span> (1999) SKP2 is required for ubiquitin-mediated degradation of the CDK inhibitor p27. Nat Cell Biol  1: 193&ndash;199. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Carrano&title=SKP2%20is%20required%20for%20ubiquitin-mediated%20degradation%20of%20the%20CDK%20inhibitor%20p27.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Muth1" id="pone.0014758-Muth1"></a><span class="authors">Muth D, Ghazaryan S, Eckerle I, Beckett E, P&ouml;hler C, et al. </span> (2010) Transcriptional repression of SKP2 is impaired in MYCN-amplified neuroblastoma. Cancer Res  70: 3791&ndash;3802. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Muth&title=Transcriptional%20repression%20of%20SKP2%20is%20impaired%20in%20MYCN-amplified%20neuroblastoma.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Raymond1" id="pone.0014758-Raymond1"></a><span class="authors">Raymond CK, Roberts BS, Garrett-Engele P, Lim LP, Johnson JM</span> (2005) Simple, quantitative primer-extension PCR assay for direct monitoring of microRNAs and short-interfering RNAs. RNA  11: 1737&ndash;1744. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Raymond&title=Simple,%20quantitative%20primer-extension%20PCR%20assay%20for%20direct%20monitoring%20of%20microRNAs%20and%20short-interfering%20RNAs.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Tibshirani1" id="pone.0014758-Tibshirani1"></a><span class="authors">Tibshirani R, Hastie T, Narasimhan B, Chu G</span> (2002) Diagnosis of multiple cancer types by shrunken centroids of gene expression. Proc Natl Acad Sci USA  99: 6567&ndash;6572. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Tibshirani&title=Diagnosis%20of%20multiple%20cancer%20types%20by%20shrunken%20centroids%20of%20gene%20expression.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Troyanskaya1" id="pone.0014758-Troyanskaya1"></a><span class="authors">Troyanskaya O, Cantor M, Sherlock G, Brown P, Hastie T, et al. </span> (2001) Missing value estimation methods for DNA microarrays. Bioinformatics  17: 520&ndash;525. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Troyanskaya&title=Missing%20value%20estimation%20methods%20for%20DNA%20microarrays.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Breitling1" id="pone.0014758-Breitling1"></a><span class="authors">Breitling R, Armengaud P, Amtmann A, Herzyk P</span> (2004) Rank products: a simple, yet powerful, new method to detect differentially regulated genes in replicated microarray experiments. FEBS Lett  573: 83&ndash;92. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Breitling&title=Rank%20products:%20a%20simple,%20yet%20powerful,%20new%20method%20to%20detect%20differentially%20regulated%20genes%20in%20replicated%20microarray%20experiments.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Edgar1" id="pone.0014758-Edgar1"></a><span class="authors">Edgar R, Domrachev M, Lash AE</span> (2002) Gene Expression Omnibus: NCBI gene expression and hybridization array data repository. Nucleic Acids Res  30: 207&ndash;210. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Edgar&title=Gene%20Expression%20Omnibus:%20NCBI%20gene%20expression%20and%20hybridization%20array%20data%20repository.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Iritani1" id="pone.0014758-Iritani1"></a><span class="authors">Iritani BM, Forbush KA, Farrar MA, Perlmutter RM</span> (1997) Control of B cell development by Ras-mediated activation of Raf. EMBO J  16: 7019&ndash;7031. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Iritani&title=Control%20of%20B%20cell%20development%20by%20Ras-mediated%20activation%20of%20Raf.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Megason1" id="pone.0014758-Megason1"></a><span class="authors">Megason SG, McMahon AP</span> (2002) A mitogen gradient of dorsal midline Wnts organizes growth in the CNS. Development  129: 2087&ndash;2098. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Megason&title=A%20mitogen%20gradient%20of%20dorsal%20midline%20Wnts%20organizes%20growth%20in%20the%20CNS.">

                    Find this article online
                  </a></li><li xpathlocation="noSelect"><a name="pone.0014758-Wu1" id="pone.0014758-Wu1"></a><span class="authors">Wu J, Bonsra AN, Du G</span> (2009) pSM155 and pSM30 vectors for miRNA and shRNA expression. Methods in Molecular Biology. United States: Springer Verlag. pp. 205&ndash;219. </li><li xpathlocation="noSelect"><a name="pone.0014758-Wrana1" id="pone.0014758-Wrana1"></a><span class="authors">Wrana JL, Attisano L, C&aacute;rcamo J, Zentella A, Doody J, et al. </span> (1992) TGF beta signals through a heteromeric protein kinase receptor complex. Cell  71: 1003&ndash;1014. <a class="find" href="http://www.plosone.org/article/findArticle.action?author=Wrana&title=TGF%20beta%20signals%20through%20a%20heteromeric%20protein%20kinase%20receptor%20complex.">

                    Find this article online
                  </a></li></ol></div>

</div>

</div>
<div style="display: none;">
<div dojotype="ambra.widget.RegionalDialog" id="AnnotationDialog" style="padding: 0pt; margin: 0pt;">
  <div class="dialog annotate">
    <div class="tipu" id="dTipu"></div>
    <div class="comment">
      <h5><span>Post Your Note (For Public Viewing)</span></h5>
      <div class="posting pane">
        <form name="createAnnotation" id="createAnnotation" method="post" action="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0014758">
          <input name="target" value="info:doi/10.1371/journal.pone.0014758" type="hidden">
          <input name="startPath" value="" type="hidden">
          <input name="startOffset" value="" type="hidden">
          <input name="endPath" value="" type="hidden">
          <input name="endOffset" value="" type="hidden">
          <input name="commentTitle" id="commentTitle" value="" type="hidden">
          <input name="comment" id="commentArea" value="" type="hidden">
          <input name="ciStatement" id="statementArea" value="" type="hidden">
          <input name="isCompetingInterest" id="isCompetingInterest" value="false" type="hidden">
          <input name="noteType" id="noteType" value="" type="hidden">
          <fieldset>
            <legend>Compose Your Note</legend>
            <span id="submitMsg" class="error" style="display: none;"></span>
            <table class="layout">
              <tbody><tr>
                <td>
                  <label for="cNoteType">This is a </label><select name="cNoteType" id="cNoteType"><option value="note">note</option><option value="correction">correction</option></select>
                  <span id="cdls" style="visibility: hidden; margin-left: 0.3em; white-space: nowrap;"><a href="http://www.plosone.org/static/commentGuidelines.action?target=info%3Adoi%2F10.1371%2Fjournal.pone.0014758#corrections">What are corrections?</a></span>
                  <label for="cTitle"><span class="none">Enter your note title</span><!-- error message text <em>A title is required for all public notes</em>--></label>
                  <input name="cTitle" id="cTitle" value="Enter your note title..." class="title" alt="Enter your note title..." type="text">
                  <label for="cArea"><span class="none">Enter your note</span><!-- error message text <em>Please enter your note</em>--></label>
                  <textarea name="cArea" id="cArea" value="Enter your note..." alt="Enter your note...">Enter your note...</textarea>
                  <input name="isPublic" value="true" type="hidden">
                </td>
                <td>&nbsp;</td>
                <td class="coi">
                  <fieldset>
                    <legend>Declare any competing interests.</legend>
                    <ul>
                      <li><label><input id="isCompetingInterestNo" checked="checked" name="competingInterest" value="false" type="radio"> No, I don't have any competing interests to declare.</label></li>
                      <li><label><input id="isCompetingInterestYes" name="competingInterest" value="true" type="radio"> Yes, I have competing interests to declare (enter below):</label></li>
                    </ul>
                    <textarea name="ciStatementArea" id="ciStatementArea" disabled="disabled" value="Enter your competing interests..." alt="Enter your competing interests...">Enter your competing interests...</textarea>
                  </fieldset>
                </td>
              </tr>
              <tr>
                <td colspan="3" class="buttons">
                  <input value="Cancel" title="Click to close and cancel" id="btn_cancel" type="button">
                  <input value="Submit" title="Click to post your note publicly" id="btn_post" class="primary" type="button">
                </td>
              </tr>
            </tbody></table>
          </fieldset>
        </form>
      </div>
    </div>
    <div class="tip" id="dTip"></div>
  </div>
</div><div dojotype="ambra.widget.ContextAction" id="ContextActionDialog" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="caTipu"></div>
    <div class="contextActionContent">
      <h5><img src="tooltip_addannotation.gif"> Add a note to this text.</h5>
      Please follow our <a href="http://www.plosone.org/static/commentGuidelines.action">guidelines for notes and comments</a> and review our <a href="http://www.plosone.org/static/competing.action">competing interests policy</a>. Comments that do not conform to our guidelines will be promptly removed and the user account disabled. The following must be avoided:
      <ul>
        <li>Remarks that could be interpreted as allegations of misconduct</li>
        <li>Unsupported assertions or statements</li>
        <li>Inflammatory or insulting language</li>
      </ul>
      <form name="contextActionForm" id="contextActionForm" class="clearfix buttons" method="post" action="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0014758">
        <input name="Continue" value="Continue" id="ContextActionDialogContinueButton" onmouseup="ambra.displayAnnotationContext.startComment(event);" title="Add a note to this text" class="primary" type="button">
        <input name="Cancel" value="Cancel" id="ContextActionDialogCancelButton" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);" title="Close this Window" type="button">
      </form>
    </div>
    <div class="tip" id="caTip"></div>
  </div>
</div>
<div dojotype="ambra.widget.ContextAction" id="ContextActionDialogNotLogged" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="canlTipu"></div>
    <div class="contextActionContent">
      <h5><img src="tooltip_addannotation.gif"> Add a note to this text.</h5>
      You must be logged in to add a note to an article.
      You may log in by <a onmousedown="ambra.displayAnnotationContext.disconnect(event);" href="http://www.plosone.org/user/secure/secureRedirect.action?goTo=%2Farticle%2Finfo%3Adoi%2F10.1371%2Fjournal.pone.0014758">clicking here</a> or <a href="#" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);">cancel this note</a>.
    </div>
    <div class="tip" id="canlTip"></div>
  </div>
</div>
<div dojotype="ambra.widget.ContextAction" id="ContextActionDialogBadSelection" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="canBDTipu"></div>
    <div class="contextActionContent">
      <h5 class="annotation icon"><img src="tooltip_addannotation.gif"> Add a note to this text.</h5>
      You cannot annotate this area of the document. <a href="#" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);">Close</a>
    </div>
    <div class="tip" id="canBDTip"></div>
  </div>
</div>
<div dojotype="ambra.widget.ContextAction" id="ContextActionDialogBadRangeSelection" class="contextActionDialog">
  <div class="dialog context">
    <div class="tipu" id="canbrTipu"></div>
    <div class="contextActionContent">
      <h5><img src="tooltip_addannotation.gif"> Add a note to this text.</h5>
      You cannot create an annotation that spans different sections of the document; please adjust your selection.<br>
      <a href="#" onclick="return false;" onmouseup="ambra.displayAnnotationContext.cancelContext(event);">Close</a>
    </div>
    <div class="tip" id="canbrTip"></div>
  </div>
</div>
<div dojotype="ambra.widget.RegionalDialog" id="CommentDialog" style="padding: 0pt; margin: 0pt;">
  <div class="dialog preview">
    <div class="tipu" id="cTipu"></div>
    <div class="btn close" id="btn_close" title="Click to close"><a title="Click to close">Close</a></div>
    <div id="cmtContainer" class="comment">
      <h6 id="viewCmtTitle"></h6>
      <div class="detail" id="viewCmtDetail"></div>
      <div class="contentwrap" id="viewComment"></div>
      <div class="contentwrap" id="viewCIStatement"></div>
      <div class="detail" id="viewLink">
        <!--<a href="#" class="commentary icon" title="Click to view full thread and respond">View all responses</a>
        <a href="#" class="respond tooltip" title="Click to respond to this posting">Respond to this</a>-->
      </div>
    </div>
    <div class="tip" id="cTip"></div>
  </div>
</div>
<div dojotype="ambra.widget.RegionalDialog" id="CommentDialogMultiple" style="padding: 0pt; margin: 0pt;">
  <div class="dialog multiple preview">
    <div class="tipu" id="mTipu"></div>
    <div class="btn close" id="btn_close_multi" title="Click to close"><a title="Click to close">Close</a></div>
    <ol id="multilist"></ol>
    <br>
    <div id="multidetail"></div>
    <div class="tip" id="mTip"></div>
  </div>
</div>


<div dojotype="dijit.Dialog" id="Rating">
  <div class="dialog annotate">
    <div class="tipu" id="dTipu"></div>
    <div class="comment">
      <h5><span>Rate This Article</span></h5>
      <div class="instructions">Please follow our <a href="http://www.plosone.org/static/ratingGuidelines.action">guidelines for rating</a> and review our <a href="http://www.plosone.org/static/competing.action">competing interests policy</a>. Comments that do not conform to our guidelines will be promptly removed and the user account disabled. The following must be avoided:
        <ol>
          <li>Remarks that could be interpreted as allegations of misconduct</li>
          <li>Unsupported assertions or statements</li>
          <li>Inflammatory or insulting language</li>
        </ol>
      </div>
      <div class="posting pane">
        <form name="ratingForm" id="ratingForm" method="post" action="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0014758">
          <input name="articleURI" value="info:doi/10.1371/journal.pone.0014758" type="hidden">  
          <input name="commentTitle" id="commentTitle" value="" type="hidden">
          <input name="comment" id="commentArea" value="" type="hidden">
          <input name="ciStatement" id="statementArea" value="" type="hidden">
          <input name="isCompetingInterest" id="isCompetingInterest" value="" type="hidden">
          <fieldset>
            <legend>Compose Your Annotation</legend>
            <span id="submitRatingMsg" class="error" style="display: none;"></span>
            <table class="layout">
              <tbody><tr>
                <td rowspan="2">
                  <label for="insight">Insight</label>
                  <ul class="star-rating rating edit" title="Rate insight" id="rateInsight">
                    <li class="current-rating pct0"></li>
                    <li><a href="javascript:void(0);" title="Bland" class="one-star" onclick="ambra.rating.setRatingCategory(this, 'insight', 1);">1</a></li>
                    <li><a href="javascript:void(0);" title="" class="two-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 2);">2</a></li>
                    <li><a href="javascript:void(0);" title="" class="three-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 3);">3</a></li>
                    <li><a href="javascript:void(0);" title="" class="four-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 4);">4</a></li>
                    <li><a href="javascript:void(0);" title="Profound" class="five-stars" onclick="ambra.rating.setRatingCategory(this, 'insight', 5);">5</a></li>
                  </ul>    
                  <input name="insight" title="insight" value="" type="hidden">
                  <label for="reliability">Reliability</label>
                  <ul class="star-rating rating edit" title="Rate reliability" id="rateReliability">
                    <li class="current-rating pct0"></li>
                    <li><a href="javascript:void(0);" title="Tenuous" class="one-star" onclick="ambra.rating.setRatingCategory(this, 'reliability', 1);">1</a></li>
                    <li><a href="javascript:void(0);" title="" class="two-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 2);">2</a></li>
                    <li><a href="javascript:void(0);" title="" class="three-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 3);">3</a></li>
                    <li><a href="javascript:void(0);" title="" class="four-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 4);">4</a></li>
                    <li><a href="javascript:void(0);" title="Unassailable" class="five-stars" onclick="ambra.rating.setRatingCategory(this, 'reliability', 5);">5</a></li>
                  </ul>    
                  <input name="reliability" title="reliability" value="" type="hidden">
                  <label for="style">Style</label>
                  <ul class="star-rating rating edit" title="Rate style" id="rateStyle">
                    <li class="current-rating pct0"></li>
                    <li><a href="javascript:void(0);" title="Crude" class="one-star" onclick="ambra.rating.setRatingCategory(this, 'style', 1);">1</a></li>
                    <li><a href="javascript:void(0);" title="" class="two-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 2);">2</a></li>
                    <li><a href="javascript:void(0);" title="" class="three-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 3);">3</a></li>
                    <li><a href="javascript:void(0);" title="" class="four-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 4);">4</a></li>
                    <li><a href="javascript:void(0);" title="Elegant" class="five-stars" onclick="ambra.rating.setRatingCategory(this, 'style', 5);">5</a></li>
                  </ul>    
                  <input name="style" title="style" value="" type="hidden">
                  <label for="cTitle"><span class="none">Enter your comment title</span><!-- error message text <em>A title is required for all public annotations</em>--></label>
                  <input name="cTitle" id="cTitle" value="Enter your comment title..." class="title" alt="Enter your comment title..." type="text">
                  <label for="cArea"><span class="none">Enter your comment</span><!-- error message text <em>Please enter your annotation</em>--></label>
                  <textarea name="cArea" id="cArea" value="Enter your comment..." alt="Enter your comment...">Enter your comment...</textarea>
                </td>
                <td rowspan="2">&nbsp;</td>
                <td class="coi">
                  <fieldset>
                    <legend>Declare any competing interests.</legend>
                    <ul>
                      <li><label><input id="isCompetingInterestNo" name="competingInterest" value="false" type="radio"> No, I don't have any competing interests to declare.</label></li>
                      <li><label><input id="isCompetingInterestYes" name="competingInterest" value="true" type="radio"> Yes, I have competing interests to declare (enter below):</label></li>
                    </ul>
                    <textarea name="ciStatementArea" id="ciStatementArea" disabled="disabled" value="Enter your competing interests..." title="Enter your competing interests...">Enter your competing interests...</textarea>
                  </fieldset>
                </td>
              </tr>
              <tr>
                <td class="buttons">
                  <input value="Cancel" title="Click to close and cancel" id="btn_cancel_rating" type="button">
                  <input value="Submit" title="Click to post your annotation publicly" id="btn_post_rating" class="primary" type="button">
                </td>
              </tr>
            </tbody></table>
          </fieldset>
        </form>
      </div>
    </div>
  </div>
</div>
<div dojotype="ambra.widget.LoadingCycle" id="LoadingCycle" class="loadingCycler">
  <img src="loading.gif" title="Loading..." height="58" width="58">
</div>
</div>
</div><!-- end : main contents -->

</div>
<!-- end : container -->	

<!-- begin : footer -->
<div id="ftr">
  <p><span>All site content, except where otherwise noted, is licensed under a <a href="http://creativecommons.org/licenses/by/2.5/" title="Creative Commons Attribution License 2.5" tabindex="200">Creative Commons Attribution License</a>.</span></p>
  <ul>
    <li><a href="http://www.plosone.org/static/privacy.action" title="PLoS Privacy Statement" tabindex="501">Privacy Statement</a></li>
    <li><a href="http://www.plosone.org/static/terms.action" title="PLoS Terms of Use" tabindex="502">Terms of Use</a></li>
    <li><a href="http://www.plos.org/advertise/" title="Advertise With PLoS" tabindex="503">Advertise</a></li>
    <li><a href="http://www.plos.org/journals/embargopolicy.html" title="PLoS Embargo Policy" tabindex="504">Media Inquiries</a></li>
    <li><a href="http://www.plos.org/journals/print.html" title="PLoS in Print" tabindex="505">PLoS in Print</a></li>
    <li><a href="http://www.plosone.org/static/sitemap.action" title="Site Map" tabindex="506">Site Map</a></li>
    <li><a href="http://www.plos.org/" title="PLoS.org" tabindex="507">PLoS.org</a></li>
  </ul>
  <div class="powered">
  <ul>
    <li><a href="http://www.plosone.org/static/releaseNotes.action" title="Ambra | Release Notes">Ambra 1.1 </a></li>
    <li>Managed Colocation provided by <a href="http://www.isc.org/" title="Internet Systems Consortium">Internet Systems Consortium</a>.</li>
  </ul>
  </div>
</div>
<!-- end : footer -->
<script type="text/javascript">

  var _namespace="";
    var loggedIn = false;

  var almHost = "http://alm.plos.org";
  var pubGetHost = "http://pubget.com/widgetizer/links_hash";
  var appContext = "";

  // Safari v3.1.1 "console.debug" issue (http://trac.dojotoolkit.org/ticket/6849) workaround
  if (/3[\.0-9]+ Safari/.test(navigator.appVersion)) {
    window.console = {
      origConsole: window.console,
      log: function(s){ this.origConsole.log(s); },
      info: function(s){ this.origConsole.info(s); },
      error: function(s){ this.origConsole.error(s); },
      warn: function(s){ this.origConsole.warn(s); }
    };
  }

  var djConfig = {
    // don't debug for IE - as dojo's firebug lite module is error prone in IE
    isDebug: false,
    parseOnLoad: true
  };
</script>
<script type="text/javascript" src="dojo.js"></script>
<script type="text/javascript" src="ambra.js"></script>
<script type="text/javascript" src="init_global.js"></script>
<script type="text/javascript" src="init_article.js"></script>
<script type="text/javascript" src="init_ratings.js"></script>
<script type="text/javascript" src="init_article_body.js"></script>
<script type="text/javascript" src="init_article_rhc.js"></script>
<script type="text/javascript" src="alm.js"></script>
<script type="text/javascript" src="articleviewscumulative.js"></script>

<script type="text/javascript">

var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">

var pageTracker = _gat._getTracker("UA-338393-1");
pageTracker._trackPageview();
pageTracker._setDomainName("www.plosone.org");
</script>

<script type="text/javascript">

var _sf_async_config={uid:16579,domain:"plosone.org"};
(function(){
  function loadChartbeat() {
    window._sf_endpt=(new Date()).getTime();
    var e = document.createElement('script');
    e.setAttribute('language', 'javascript');
    e.setAttribute('type', 'text/javascript');
    e.setAttribute('src',
       (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
       "js/chartbeat.js");
    document.body.appendChild(e);
  }
  var oldonload = window.onload;
  window.onload = (typeof window.onload != 'function') ?
     loadChartbeat : function() { oldonload(); loadChartbeat(); };
})();

</script>

</body>
</html>
